<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
	<title>  Transitions from Single- to Multi-Locus approach in determining cryptic and describing new species within  Beauveria  genus (Cordycipitaceae, Hypocreales): a review   | ABDESSAMAD | Phytotaxa</title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"VgMHUldUABAEVVdWAAADUlE=",licenseKey:"4f72ad4ec1",applicationID:"416471445"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var i=e[n]={exports:{}};t[n][0].call(i.exports,function(e){var i=t[n][1][e];return r(i||e)},i,i.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var i=0;i<n.length;i++)r(n[i]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var i,o=t("ee"),a=t(23),c={};try{i=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,i.indexOf("dev")!==-1&&(c.dev=!0),i.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&o.on("internal-error",function(t){r(t.stack)}),c.dev&&o.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{p?p-=1:i(c||new UncaughtException(t,e,n),!0)}catch(f){try{o("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function i(t,e){var n=e?null:s.now();o("err",[t,n])}var o=t("handle"),a=t(24),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,l="nr@seenError",p=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(9),t(8),"addEventListener"in window&&t(5),s.xhrWrappable&&t(10),d=!0)}c.on("fn-start",function(t,e,n){d&&(p+=1)}),c.on("fn-err",function(t,e,n){d&&!n[l]&&(f(n,l,function(){return!0}),this.thrown=!0,i(n))}),c.on("fn-end",function(){d&&!this.thrown&&p>0&&(p-=1)}),c.on("internal-error",function(t){o("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var i=t("ee"),o=t("handle"),a=t(9),c=t(8),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",l="resource",p="-start",h="-end",m="fn"+p,w="fn"+h,v="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(7),"addEventListener"in window&&t(5);var x=NREUM.o.EV;i.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),i.on(w,function(t,e){var n=t[0];n instanceof x&&o("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(w,function(t,e){o(v,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(w,function(t,e){o(v,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),i.on(g+p,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),i.on(g+h,function(t){o("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){o(d,[window.performance.getEntriesByType(l)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){o(d,[window.performance.getEntriesByType(l)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],5:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&i(e)}function i(t){c.inPlace(t,[u,d],"-",o)}function o(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(i(window),i(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],6:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=o(arguments),e={};i.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return i.emit(n+"start",[t,a],c),c.then(function(t){return i.emit(n+"end",[null,t],c),t},function(t){throw i.emit(n+"end",[t],c),t})})}var i=t("ee").get("fetch"),o=t(24),a=t(23);e.exports=i;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,l=c.Response,p=c.fetch,h="prototype",m="nr@context";d&&l&&p&&(a(u,function(t,e){r(d[h],e,f),r(l[h],e,f)}),r(c,"fetch",s),i.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),i.emit(s+"done",[null,e],n)}else i.emit(s+"done",[t],n)}))},{}],7:[function(t,e,n){var r=t("ee").get("history"),i=t("wrap-function")(r);e.exports=r;var o=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;o&&o.pushState&&o.replaceState&&(a=o),i.inPlace(a,["pushState","replaceState"],"-")},{}],8:[function(t,e,n){var r=t("ee").get("raf"),i=t("wrap-function")(r),o="equestAnimationFrame";e.exports=r,i.inPlace(window,["r"+o,"mozR"+o,"webkitR"+o,"msR"+o],"raf-"),r.on("raf-start",function(t){t[0]=i(t[0],"fn-")})},{}],9:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function i(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var o=t("ee").get("timer"),a=t("wrap-function")(o),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=o,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),o.on(s+u,r),o.on(c+u,i)},{}],10:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function i(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function o(t){y.push(t),h&&(b?b.then(a):w?w(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(5);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),l=NREUM.o,p=l.XHR,h=l.MO,m=l.PR,w=l.SI,v="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new p(t);try{u.emit("new-xhr",[e],e),e.addEventListener(v,i,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(p,x),x.prototype=p.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),o(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!w&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===v||a()})},{}],11:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=p.generateSpanId(),m=p.generateTraceId(),w=Date.now(),v={spanId:h,traceId:m,timestamp:w};return(t.sameOrigin||s(t)&&l())&&(v.traceContextParentHeader=i(h,m),v.traceContextStateHeader=o(h,w,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(v.newrelicHeader=a(h,m,w,n,r,f)),v}function i(t,e){return"00-"+e+"-"+t+"-01"}function o(t,e,n,r,i){var o=0,a="",c=1,s="",f="";return i+"@nr="+o+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,i,o){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:i,id:t,tr:e,ti:n}};return o&&r!==o&&(c.d.tk=o),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var i=h(n.allowed_origins[r]);if(t.hostname===i.hostname&&t.protocol===i.protocol&&t.port===i.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function l(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var p=t(20),h=t(13);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],12:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<l;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):o(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function i(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function o(t,e){t.params.status=e.status;var n=w(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(13),f=t(11).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],l=d.length,p=t("id"),h=t(17),m=t(16),w=t(14),v=window.XMLHttpRequest;a.features.xhr=!0,t(10),t(6),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){o(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],i=this;if(n&&r){var o=m(r);o&&(n.txSize=o)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||i.loadCaptureCalled||(i.params.aborted=!0),("load"!==t.type||i.called===i.totalCbs&&(i.onloadCalled||"function"!=typeof e.onload))&&i.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<l;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof v&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof v&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof v&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var i=f(this.parsedOrigin);if(i&&(i.newrelicHeader||i.traceContextParentHeader))if("string"==typeof t[0]){var o={};for(var a in r)o[a]=r[a];o.headers=new Headers(r.headers||{}),e(o.headers,i)&&(this.dt=i),t.length>1?t[1]=o:t.push(o)}else t[0]&&t[0].headers&&e(t[0].headers,i)&&(this.dt=i)})}},{}],13:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,i={};e.href=t,i.port=e.port;var o=e.href.split("://");!i.port&&o[1]&&(i.port=o[1].split("/")[0].split("@").pop().split(":")[1]),i.port&&"0"!==i.port||(i.port="https"===o[0]?"443":"80"),i.hostname=e.hostname||n.hostname,i.pathname=e.pathname,i.protocol=o[0],"/"!==i.pathname.charAt(0)&&(i.pathname="/"+i.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return i.sameOrigin=a&&(!e.hostname||c),"/"===i.pathname&&(r[t]=i),i}},{}],14:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?i(t.response):"text"===n||""===n||void 0===n?i(t.responseText):void 0}var i=t(16);e.exports=r},{}],15:[function(t,e,n){function r(){}function i(t,e,n){return function(){return o(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var o=t("handle"),a=t(23),c=t(24),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",p=l+"ixn-";a(d,function(t,e){u[e]=i(l+e,!0,"api")}),u.addPageAction=i(l+"addPageAction",!0),u.setCurrentRouteName=i(l+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,i="function"==typeof e;return o(p+"tracer",[f.now(),t,n],r),function(){if(s.emit((i?"":"no-")+"fn-start",[f.now(),r,i],n),i)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=i(p+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),o("err",[t,f.now(),!1,e])}},{}],16:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],17:[function(t,e,n){var r=0,i=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);i&&(r=+i[1]),e.exports=r},{}],18:[function(t,e,n){function r(){return c.exists&&performance.now?Math.round(performance.now()):(o=Math.max((new Date).getTime(),o))-a}function i(){return o}var o=(new Date).getTime(),a=o,c=t(25);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=i},{}],19:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function i(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function o(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!w){var e=Math.round(t.timeStamp),n={type:t.type};e<=l.now()?n.fid=l.now()-e:e>l.offset&&e<=Date.now()?(e-=l.offset,n.fid=l.now()-e):e=l.now(),w=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[l.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),l=t("loader"),p=t(22),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(i);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(o);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var w=!1,v=["click","keydown","mousedown","pointerdown","touchstart"];v.forEach(function(t){document.addEventListener(t,a,!1)})}p(c)}},{}],20:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var i,o="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<o.length;c++)i=o[c],"x"===i?a+=t().toString(16):"y"===i?(i=3&t()|8,a+=i.toString(16)):a+=i;return a}function i(){return a(16)}function o(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,i=window.crypto||window.msCrypto;i&&i.getRandomValues&&Uint8Array&&(n=i.getRandomValues(new Uint8Array(31)));for(var o=[],a=0;a<t;a++)o.push(e().toString(16));return o.join("")}e.exports={generateUuid:r,generateSpanId:i,generateTraceId:o}},{}],21:[function(t,e,n){function r(t,e){if(!i)return!1;if(t!==i)return!1;if(!e)return!0;if(!o)return!1;for(var n=o.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var i=null,o=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(i="Safari",o=s[1])}e.exports={agent:i,version:o,match:r}},{}],22:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[i]?"hidden":"visible")}"addEventListener"in document&&o&&document.addEventListener(o,e,!1)}e.exports=r;var i,o,a;"undefined"!=typeof document.hidden?(i="hidden",o="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",o="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",o="webkitvisibilitychange",a="webkitVisibilityState")},{}],23:[function(t,e,n){function r(t,e){var n=[],r="",o=0;for(r in t)i.call(t,r)&&(n[o]=e(r,t[r]),o+=1);return n}var i=Object.prototype.hasOwnProperty;e.exports=r},{}],24:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,i=n-e||0,o=Array(i<0?0:i);++r<i;)o[r]=t[e+r];return o}e.exports=r},{}],25:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function i(t){function e(t){return t&&t instanceof r?t:t?s(t,c,o):o()}function n(n,r,i,o){if(!l.aborted||o){t&&t(n,r,i);for(var a=e(i),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function p(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function w(t){return d[t]=d[t]||i(n)}function v(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:p,addEventListener:p,removeEventListener:h,emit:n,get:w,listeners:m,context:e,buffer:v,abort:a,aborted:!1};return x}function o(){return new r}function a(){(u.api||u.feature)&&(l.aborted=!0,u=l.backlog={})}var c="nr@context",s=t("gos"),f=t(23),u={},d={},l=e.exports=i();l.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(i.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(o){}return t[e]=r,r}var i=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){i.buffer([t],r),i.emit(t,e,n)}var i=t("ee").get("handle");e.exports=r,r.ee=i},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,o,function(){return i++})}var i=1,o="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!b++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();s(g,function(e,n){t[e]||(t[e]=n)});var n=a();c("mark",["onload",n+x.offset],null,"api"),c("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function i(){"complete"===l.readyState&&o()}function o(){c("mark",["domContent",a()+x.offset],null,"api")}var a=t(18),c=t("handle"),s=t(23),f=t("ee"),u=t(21),d=window,l=d.document,p="addEventListener",h="attachEvent",m=d.XMLHttpRequest,w=m&&m.prototype;NREUM.o={ST:setTimeout,SI:d.setImmediate,CT:clearTimeout,XHR:m,REQ:d.Request,EV:d.Event,PR:d.Promise,MO:d.MutationObserver};var v=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1184.min.js"},y=m&&w&&w[p]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:a.getLastTimestamp(),now:a,origin:v,features:{},xhrWrappable:y,userAgent:u};t(15),t(19),l[p]?(l[p]("DOMContentLoaded",o,!1),d[p]("load",r,!1)):(l[h]("onreadystatechange",i),d[h]("onload",r)),c("mark",["firstbyte",a.getLastTimestamp()],null,"api");var b=0},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var i=t("ee"),o=t(24),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,i){function nrWrapper(){var r,a,c,s;try{a=this,r=o(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){l([f,"",[r,a,i],c])}u(e+"start",[r,a,i],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,i,o){i||(i="");var a,c,s,f="-"===i.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+i:i,o,c))}function u(n,r,i){if(!s||e){var o=s;s=!0;try{t.emit(n,r,i,e)}catch(a){l([a,n,r,i])}s=o}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){l([r])}for(var i in t)c.call(t,i)&&(e[i]=t[i]);return e}function l(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=i),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,12,4,3]);</script>
	<meta name="description" content="  Transitions from Single- to Multi-Locus approach in determining cryptic and describing new species within  Beauveria  genus (Cordycipitaceae, Hypocreales): a review  " />
			<meta name="keywords" content="Hypocreales, DNA, Taxonomy, Multi-locus approach, GCPSR, Species Circumscription, Fungi" />
	
	
	<link rel="schema.DC" href="http://purl.org/dc/elements/1.1/" />

	<meta name="DC.Contributor.Sponsor" xml:lang="en" content=""/>
	<meta name="DC.Creator.PersonalName" content="IMOULAN ABDESSAMAD"/>
	<meta name="DC.Date.created" scheme="ISO8601" content="2019-08-05"/>
	<meta name="DC.Date.dateSubmitted" scheme="ISO8601" content="2019-02-25"/>
	<meta name="DC.Date.issued" scheme="ISO8601" content="2019-08-05"/>
	<meta name="DC.Date.modified" scheme="ISO8601" content="2019-08-05"/>
	<meta name="DC.Description" xml:lang="en" content="  Beauveria  is one of the most prominent genus within entomopathogenic fungi. This genus is an asexually reproducing fungus which includes morphologically indistinguishable species. The delimitation of species in  Beauveria  is controversial due to the lack of distinctive taxonomic features and the extensive overlap in the shape and the sizes of conidia. Moreover, cryptic species have been revealed in  Beauveria  using molecular tools making species identification a challenge within the genus. Since 1990s, the development of reliable molecular tools based on DNA sequences has become increasingly useful for species delineation and recognition allowing a best application of genealogical concordance phylogenetic species recognition. Actually, specimen’s taxonomy becomes tightly dependent on DNA based-data. Single-gene based on the internal transcribed spacers (ITS) region is the common barcoding DNA, but has reached its limit in delimiting closely related species, including  Beauveria  species. With the advances in sequencing technology, the focus has been shifted from single to multiple locus data to identify fungal isolates adapting genealogical concordance phylogenetic species recognition (GCPSR) criterion. Multi-locus approach has been increasingly more applied in assigning fungal samples to a given species and significantly improves  Beauveria  taxonomy raising the number of  Beauveria  species to heighten species. "/>
	<meta name="DC.Format" scheme="IMT" content="application/pdf"/>
	<meta name="DC.Identifier" content="phytotaxa.413.4.1"/>
	<meta name="DC.Identifier.pageNumber" content="257–273"/>
							<meta name="DC.Identifier.DOI" content="10.11646/phytotaxa.413.4.1"/>
		<meta name="DC.Identifier.URI" content="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.413.4.1"/>
	<meta name="DC.Language" scheme="ISO639-1" content="en"/>
	<meta name="DC.Rights" content="Copyright (c) 2019 Phytotaxa" />
	<meta name="DC.Rights" content=""/>
	<meta name="DC.Source" content="Phytotaxa"/>
	<meta name="DC.Source.ISSN" content="1179-3163"/>
	<meta name="DC.Source.Issue" content="4"/>	<meta name="DC.Source.URI" content="https://www.biotaxa.org/Phytotaxa"/>
	<meta name="DC.Source.Volume" content="413"/>						<meta name="DC.Subject" xml:lang="en" content="Hypocreales, DNA, Taxonomy, Multi-locus approach, GCPSR, Species Circumscription, Fungi"/>
				<meta name="DC.Title" content="  Transitions from Single- to Multi-Locus approach in determining cryptic and describing new species within  Beauveria  genus (Cordycipitaceae, Hypocreales): a review  "/>
		<meta name="DC.Type" content="Text.Serial.Journal"/>
	<meta name="DC.Type.articleType" content="Article"/>
		<meta name="gs_meta_revision" content="1.1" />
	<meta name="citation_journal_title" content="Phytotaxa"/>
	<meta name="citation_issn" content="1179-3163"/>
        <meta name="citation_author" content="IMOULAN ABDESSAMAD"/>
        <meta name="citation_author_institution" content="Department of Biology, Faculty of Science and Technics of Errachidia, Mouly Ismail University, 50050, B.P. 298, Meknes, Morocco"/>
<meta name="citation_title" content="  Transitions from Single- to Multi-Locus approach in determining cryptic and describing new species within  Beauveria  genus (Cordycipitaceae, Hypocreales): a review  "/>

	<meta name="citation_date" content="2019/08/05"/>

	<meta name="citation_volume" content="413"/>
	<meta name="citation_issue" content="4"/>

			<meta name="citation_firstpage" content="257"/>
				<meta name="citation_lastpage" content="273"/>
								<meta name="citation_doi" content="10.11646/phytotaxa.413.4.1"/>
		<meta name="citation_abstract_html_url" content="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.413.4.1"/>
	<meta name="citation_language" content="en"/>
						<meta name="citation_keywords" xml:lang="en" content="Hypocreales, DNA, Taxonomy, Multi-locus approach, GCPSR, Species Circumscription, Fungi"/>
									<meta name="citation_pdf_url" content="https://www.biotaxa.org/Phytotaxa/article/download/phytotaxa.413.4.1/49904"/>
				

	<link rel="stylesheet" href="https://www.biotaxa.org/lib/pkp/styles/pkp.css" type="text/css" />
	<link rel="stylesheet" href="https://www.biotaxa.org/lib/pkp/styles/common.css" type="text/css" />
	<link rel="stylesheet" href="https://www.biotaxa.org/styles/common.css" type="text/css" />
	<link rel="stylesheet" href="https://www.biotaxa.org/styles/compiled.css" type="text/css" />
	<link rel="stylesheet" href="https://www.biotaxa.org/styles/articleView.css" type="text/css" />
			<link rel="stylesheet" href="https://www.biotaxa.org/lib/pkp/styles/rtEmbedded.css" type="text/css" />
	
	
	
	<link rel="stylesheet" href="https://www.biotaxa.org/styles/sidebar.css" type="text/css" />		<link rel="stylesheet" href="https://www.biotaxa.org/styles/rightSidebar.css" type="text/css" />	
			<link rel="stylesheet" href="https://www.biotaxa.org/public/site/sitestyle.css" type="text/css" />
			<link rel="stylesheet" href="https://www.biotaxa.org/plugins/blocks/languageToggle/styles/languageToggle.css" type="text/css" />
	
	<!-- Base Jquery -->
	<script type="text/javascript" src="//www.google.com/jsapi"></script>
	<script type="text/javascript">
		// Provide a local fallback if the CDN cannot be reached
		if (typeof google == 'undefined') {
			document.write(unescape("%3Cscript src='https://www.biotaxa.org/lib/pkp/js/lib/jquery/jquery.min.js' type='text/javascript'%3E%3C/script%3E"));
			document.write(unescape("%3Cscript src='https://www.biotaxa.org/lib/pkp/js/lib/jquery/plugins/jqueryUi.min.js' type='text/javascript'%3E%3C/script%3E"));
		} else {
			google.load("jquery", "1.4.4");
			google.load("jqueryui", "1.8.6");
		}
	</script>
	
	<!-- Compiled scripts -->
			
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/lib/jquery/plugins/jquery.tag-it.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/lib/jquery/plugins/jquery.cookie.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/functions/fontController.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/functions/general.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/functions/jqueryValidatorI18n.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/Helper.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/ObjectProxy.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/Handler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/linkAction/LinkActionRequest.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/features/Feature.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/SiteHandler.js"></script><!-- Included only for namespace definition -->
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/UrlInDivHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/AutocompleteHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/ExtrasOnDemandHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/form/FormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/form/AjaxFormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/form/ClientFormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/grid/GridHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/linkAction/LinkActionHandler.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/js/pages/search/SearchFormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/js/statistics/ReportGeneratorFormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/plugins/generic/lucene/js/LuceneAutocompleteHandler.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/lib/jquery/plugins/jquery.pkp.js"></script>	
	
	<script type="text/javascript" src="https://www.biotaxa.org/js/relatedItems.js"></script>
	<script type="text/javascript" src="https://www.biotaxa.org/js/inlinePdf.js"></script>
	<script type="text/javascript" src="https://www.biotaxa.org/js/pdfobject.js"></script>

</head>
<body id="pkp-common-openJournalSystems">

<div id="container">

<div id="header">
<div id="headerTitle">
<h1>
	<img src="https://www.biotaxa.org/public/journals/2/pageHeaderTitleImage_en_US.jpg" width="1101" height="98" alt="Page Header" />
</h1>
</div>
</div>

<div id="body">

	<div id="sidebar">
							<div id="rightSidebar">
				<div class="block" id="sidebarDevelopedBy">
	<a class="blockTitle" href="http://pkp.sfu.ca/ojs/" id="developedBy">Open Journal Systems</a>
</div><div class="block" id="sidebarHelp">
	<a class="blockTitle" href="javascript:openHelp('https://www.biotaxa.org/Phytotaxa/help')">Journal Help</a>
</div><div class="block" id="sidebarSubscription">
	<span class="blockTitle">Subscription</span>
							Login to verify subscription
			</div><div class="block" id="sidebarUser">
			<span class="blockTitle">User</span>
	
												<form method="post" action="https://www.biotaxa.org/Phytotaxa/login/signIn">
					<table>
						<tr>
							<td><label for="sidebar-username">Username</label></td>
							<td><input type="text" id="sidebar-username" name="username" value="" size="12" maxlength="32" class="textField" /></td>
						</tr>
						<tr>
							<td><label for="sidebar-password">Password</label></td>
							<td><input type="password" id="sidebar-password" name="password" value="" size="12" class="textField" /></td>
						</tr>
						<tr>
							<td colspan="2"><input type="checkbox" id="remember" name="remember" value="1" /> <label for="remember">Remember me</label></td>
						</tr>
						<tr>
							<td colspan="2"><input type="submit" value="Login" class="button" /></td>
						</tr>
					</table>
				</form>
						</div> <div class="block" id="notification">
	<span class="blockTitle">Notifications</span>
	<ul>
					<li><a href="https://www.biotaxa.org/Phytotaxa/notification">View</a></li>
			<li><a href="https://www.biotaxa.org/Phytotaxa/notification/subscribeMailList">Subscribe</a></li>
			</ul>
</div>

<div class="block" id="sidebarNavigation">
	<span class="blockTitle">Journal Content</span>

	<form id="simpleSearchForm" action="https://www.biotaxa.org/Phytotaxa/search/search">
		<table id="simpleSearchInput">
			<tr>
				<td>
													<label for="simpleQuery">Search <br />
					<input type="text" id="simpleQuery" name="simpleQuery" size="15" maxlength="255" value="" class="textField" /></label>
								</td>
			</tr>
			<tr>
				<td><label for="searchField">
				Search Scope
				<br />
				<select id="searchField" name="searchField" size="1" class="selectMenu">
					<option label="All" value="query">All</option>
<option label="Authors" value="authors">Authors</option>
<option label="Title" value="title">Title</option>
<option label="Abstract" value="abstract">Abstract</option>
<option label="Index terms" value="indexTerms">Index terms</option>
<option label="Full Text" value="galleyFullText">Full Text</option>

				</select></label>
				</td>
			</tr>
			<tr>
				<td><input type="submit" value="Search" class="button" /></td>
			</tr>
		</table>
	</form>

	<br />

		<span class="blockSubtitle">Browse</span>
	<ul>
		<li><a href="https://www.biotaxa.org/Phytotaxa/issue/archive">By Issue</a></li>
		<li><a href="https://www.biotaxa.org/Phytotaxa/search/authors">By Author</a></li>
		<li><a href="https://www.biotaxa.org/Phytotaxa/search/titles">By Title</a></li>
		
					<li><a href="https://www.biotaxa.org/index">Other Journals</a></li>
						</ul>
	</div>
<div class="block" id="sidebarKeywordCloud">
	<span class="blockTitle">Keywords</span>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Algae"><span style="font-size: 92%;">Algae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Andes"><span style="font-size: 78%;">Andes</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Asteraceae"><span style="font-size: 79%;">Asteraceae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Atlantic%20Forest"><span style="font-size: 78%;">Atlantic Forest</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Basidiomycota"><span style="font-size: 78%;">Basidiomycota</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Brazil"><span style="font-size: 85%;">Brazil</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Bryophytes"><span style="font-size: 81%;">Bryophytes</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Cerrado"><span style="font-size: 77%;">Cerrado</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=China"><span style="font-size: 89%;">China</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Compositae"><span style="font-size: 82%;">Compositae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Eudicots"><span style="font-size: 175%;">Eudicots</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Fabaceae"><span style="font-size: 78%;">Fabaceae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Fungi"><span style="font-size: 105%;">Fungi</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=General"><span style="font-size: 78%;">General</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=ITS"><span style="font-size: 80%;">ITS</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=IUCN"><span style="font-size: 78%;">IUCN</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=India"><span style="font-size: 82%;">India</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Iran"><span style="font-size: 77%;">Iran</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Magnoliids"><span style="font-size: 80%;">Magnoliids</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Mexico"><span style="font-size: 78%;">Mexico</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Monocots"><span style="font-size: 118%;">Monocots</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Morphology"><span style="font-size: 77%;">Morphology</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Neotropics"><span style="font-size: 81%;">Neotropics</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Nomenclature"><span style="font-size: 79%;">Nomenclature</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Orchidaceae"><span style="font-size: 79%;">Orchidaceae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Phylogeny"><span style="font-size: 79%;">Phylogeny</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Pteridophytes"><span style="font-size: 83%;">Pteridophytes</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=SEM"><span style="font-size: 78%;">SEM</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=South%20America"><span style="font-size: 80%;">South America</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Taxonomy"><span style="font-size: 92%;">Taxonomy</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Turkey"><span style="font-size: 79%;">Turkey</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Vietnam"><span style="font-size: 78%;">Vietnam</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=biodiversity"><span style="font-size: 79%;">biodiversity</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=conservation"><span style="font-size: 79%;">conservation</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=diatoms"><span style="font-size: 78%;">diatoms</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=distribution"><span style="font-size: 78%;">distribution</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=endemic"><span style="font-size: 81%;">endemic</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=endemism"><span style="font-size: 83%;">endemism</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=flora"><span style="font-size: 78%;">flora</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=lectotype"><span style="font-size: 81%;">lectotype</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=lectotypification"><span style="font-size: 80%;">lectotypification</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=molecular%20phylogeny"><span style="font-size: 80%;">molecular phylogeny</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=morphology"><span style="font-size: 92%;">morphology</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=new%20combination"><span style="font-size: 78%;">new combination</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=new%20species"><span style="font-size: 101%;">new species</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=new%20taxon"><span style="font-size: 78%;">new taxon</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=nomenclature"><span style="font-size: 91%;">nomenclature</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=phylogeny"><span style="font-size: 91%;">phylogeny</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=systematics"><span style="font-size: 79%;">systematics</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=taxonomy"><span style="font-size: 162%;">taxonomy</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=typification"><span style="font-size: 83%;">typification</span></a>
	</div>
<!-- Add javascript required for font sizer -->
<script type="text/javascript">
	<!--
	$(function(){
		fontSize("#sizer", "body", 9, 16, 32, ""); // Initialize the font sizer
	});
	// -->
</script>

<div class="block" id="sidebarFontSize" style="margin-bottom: 4px;">
	<span class="blockTitle">Font Size</span>
	<div id="sizer"></div>
</div>
<br /><div class="block" id="sidebarInformation">
	<span class="blockTitle">Information</span>
	<ul>
		<li><a href="https://www.biotaxa.org/Phytotaxa/information/readers">For Readers</a></li>		<li><a href="https://www.biotaxa.org/Phytotaxa/information/authors">For Authors</a></li>		<li><a href="https://www.biotaxa.org/Phytotaxa/information/librarians">For Librarians</a></li>	</ul>
</div>


<div class="block" id="sidebarRTArticleTools">

	<span class="blockTitle">Article Tools</span>
				<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/printArticle.png" class="articleToolIcon" alt="" /> <a href="javascript:openRTWindow('https://www.biotaxa.org/Phytotaxa/rt/printerFriendly/phytotaxa.413.4.1/0');">Print this article</a>
		</div>
				<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/metadata.png" class="articleToolIcon" alt=""/> <a href="javascript:openRTWindow('https://www.biotaxa.org/Phytotaxa/rt/metadata/phytotaxa.413.4.1/0');">Indexing metadata</a><br />
		</div>
				<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/citeArticle.png" class="articleToolIcon" alt=""/> <a href="javascript:openRTWindow('https://www.biotaxa.org/Phytotaxa/rt/captureCite/phytotaxa.413.4.1/0');">How to cite item</a><br />
		</div>
					<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/findingReferences.png" class="articleToolIcon" alt=""/> <a href="javascript:openRTWindow('https://www.biotaxa.org/Phytotaxa/rt/findingReferences/46643/0');">Finding References</a>
		</div>
					<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/emailArticle.png" class="articleToolIcon" alt=""/>
			Email this article <span style="font-size: 0.8em">(Login required)</span>		</div>
				<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/emailArticle.png" class="articleToolIcon" alt=""/>
			Email the author <span style="font-size: 0.8em">(Login required)</span>		</div>
		</div>


<script type="text/javascript">
	initRelatedItems();
</script>


<div class="block" id="sidebarRTRelatedItems">
	<span class="blockTitle">Related Items</span>
		<table width="100%">
			<tr>
				<td valign="top" style="width:26px;">
					<img src="https://www.biotaxa.org/plugins/blocks/relatedItems/icons/lookupTerms.png" class="articleToolIcon" alt=""/>
				</td>
				<td valign="top">
					<div id="relatedItems">
						<ul class="plain">
																					<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/445');">Author&#039;s work</a></li>
																																									<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/447');">Related studies</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/448');">Book searches</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/449');">Pay-per-view</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/450');">Relevant portals</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/451');">Online forums</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/452');">Teaching files</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/453');">Government policy</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/454');">Media reports</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/455');">Web search</a></li>
																			</ul>
					</div>
					<div id="toggleRelatedItems">
						<a id="hideRelatedItems" style="display:none;" href="javascript:void(0)">
							<img src="https://www.biotaxa.org/plugins/blocks/relatedItems/icons/magnifier_zoom_out.png" alt=""/> 
							Hide
						</a>
						<a id="showRelatedItems" href="javascript:void(0)">
							<img src="https://www.biotaxa.org/plugins/blocks/relatedItems/icons/magnifier_zoom_in.png" alt=""/>
							Show all
						</a>
					</div>
				</td>
			</tr>
		</table>
</div>


<div class="block" id="sidebarRTAuthorBios">
	<span class="blockTitle">
					About The Author
			</span>
		<div class="authorBio">
	<p>
		<em>IMOULAN ABDESSAMAD</em>
								<br />Department of Biology, Faculty of Science and Technics of Errachidia, Mouly Ismail University, 50050, B.P. 298, Meknes, Morocco		<br />Morocco	</p>

	<p></p>
	</div>
	
	</div>
			</div>
			</div>

<div id="main">

<div id="navbar">
	<ul class="menu">
		<li id="home"><a href="https://www.biotaxa.org/Phytotaxa/index">Home</a></li>
		<li id="about"><a href="https://www.biotaxa.org/Phytotaxa/about">About</a></li>

					<li id="login"><a href="https://www.biotaxa.org/Phytotaxa/login">Login</a></li>
							<li id="register"><a href="https://www.biotaxa.org/Phytotaxa/user/register">Register</a></li>
												<li id="search"><a href="https://www.biotaxa.org/Phytotaxa/search">Search</a></li>
		
					<li id="current"><a href="https://www.biotaxa.org/Phytotaxa/issue/current">Current</a></li>
			<li id="archives"><a href="https://www.biotaxa.org/Phytotaxa/issue/archive">Archives</a></li>
		
					<li id="announcements"><a href="https://www.biotaxa.org/Phytotaxa/announcement">Announcements</a></li>
				

									<li class="navItem" id="navItem-0"><a href="http://www.mapress.com/phytotaxa/">Publisher</a></li>
												<li class="navItem" id="navItem-1"><a href="http://www.biotaxa.org">Biotaxa</a></li>
						</ul>
</div>
<div id="breadcrumb">
	<a href="https://www.biotaxa.org/Phytotaxa/index" target="_parent">Home</a> &gt;
	<a href="https://www.biotaxa.org/Phytotaxa/issue/view/phytotaxa.413.4" target="_parent">Vol 413, No 4</a> &gt;	<a href="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.413.4.1/0" class="current" target="_parent">ABDESSAMAD</a>
</div>

<div id="content">


	<div id="topBar">
							<div id="accessKey">
				<img src="https://www.biotaxa.org/lib/pkp/templates/images/icons/fulltext_open_medium.gif" alt="Open Access" />
				Open Access&nbsp;
				<img src="https://www.biotaxa.org/lib/pkp/templates/images/icons/fulltext_restricted_medium.gif" alt="Restricted Access" />
									Subscription Access
							</div>
			</div>
		
	<div id="articleTitle"><h3><p><strong>Transitions from Single- to Multi-Locus approach in determining cryptic and describing new species within <em>Beauveria</em> genus (Cordycipitaceae, Hypocreales): a review</strong></p></h3></div>
	<div id="authorString"><em>IMOULAN ABDESSAMAD</em></div>
	<br />
			<div id="articleAbstract">
		<h4>Abstract</h4>
		<br />
		<div><p><em>Beauveria </em>is one of the most prominent genus within entomopathogenic fungi. This genus is an asexually reproducing fungus which includes morphologically indistinguishable species. The delimitation of species in <em>Beauveria</em> is controversial due to the lack of distinctive taxonomic features and the extensive overlap in the shape and the sizes of conidia. Moreover, cryptic species have been revealed in <em>Beauveria</em> using molecular tools making species identification a challenge within the genus. Since 1990s, the development of reliable molecular tools based on DNA sequences has become increasingly useful for species delineation and recognition allowing a best application of genealogical concordance phylogenetic species recognition. Actually, specimen’s taxonomy becomes tightly dependent on DNA based-data. Single-gene based on the internal transcribed spacers (ITS) region is the common barcoding DNA, but has reached its limit in delimiting closely related species, including <em>Beauveria</em> species. With the advances in sequencing technology, the focus has been shifted from single to multiple locus data to identify fungal isolates adapting genealogical concordance phylogenetic species recognition (GCPSR) criterion. Multi-locus approach has been increasingly more applied in assigning fungal samples to a given species and significantly improves <em>Beauveria</em> taxonomy raising the number of <em>Beauveria</em> species to heighten species.</p></div>
		<br />
		</div>
	
			<div id="articleSubject">
		<h4>Keywords</h4>
		<br />
		<div>Hypocreales, DNA, Taxonomy, Multi-locus approach, GCPSR, Species Circumscription, Fungi</div>
		<br />
		</div>
	
				
			<div id="articleFullText">
		<h4>Full Text:</h4>
									<a href="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.413.4.1/49904" class="file" target="_parent">PDF/A (417KB)</a>
																			<img class="accessLogo" src="https://www.biotaxa.org/lib/pkp/templates/images/icons/fulltext_restricted_medium.gif" alt="Restricted Access" />
											</div>
	
			<div id="articleCitations">
		<h4>References</h4>
		<br />
		<div>
							<p><p>Agrawal, Y., Mual, P. &amp; Shenoy, B.D. (2014) Multi-gene genealogies reveal cryptic species <em>Beauveria rudraprayagi</em> sp. nov. from India. <em>Mycosphere</em> 5: 719–736.</p><p> https://doi.org/10.5943/mycosphere/5/6/3</p><p>Alvarez, I. &amp; Wendel, J.F. (2003) Ribosomal ITS sequences and plant phylogenetic inference. <em>Molecular Phylogenetics and Evolution</em> 29: 417–434.</p><p> https://doi.org/10.1016/S1055-7903(03)00208-2</p><p>Anderson, J.B. &amp; Ullrich, R.C. (1979) Biological species of <em>Armillaria mellea</em> in North America. <em>Mycologia</em> 71: 402–414.</p><p> https://doi.org/10.2307/3759160</p><p>Aquino De Muro, M., Elliott, S., Moore, D., Parker, B.L., Skinner, M., Reid, W. &amp; El Bouhssini, M. (2005) Molecular characterisation of <em>Beauveria bassiana</em> isolates obtained from overwintering sites of Sunn Pests (<em>Eurygaster and Aelia species</em>). <em>Mycological Research</em> 109: 294–306.</p><p> https://doi.org/10.1017/S0953756204001832</p><p>Arnold, A.E. &amp; Lutzoni, F. (2007) Diversity and host range of foliar fungal endophytes: are tropical trees biodiversity hot spots? <em>Ecology</em> 88:541–549.</p><p> https://doi.org/10.1890/05-1459</p><p>Avise, J.C. (1990) Principles of genealogical concordance in species concepts and biological taxonomy. <em>In</em>: Futuyma, D. &amp; Antonovics, J. (Eds.) <em>Oxford surveys in evolutionary biology</em>, vol 7. Oxford University Press, Oxford, pp. 45–67</p><p>Avise, J.C. &amp; Ball, R.M. (1990) Principles of genealogical concordance in species concepts and biological taxonomy. <em>Oxford Surveys in Evolutionary Biology</em> 7: 45–67.</p><p>Balajee, S.A., Houbraken, J., Verweij, P.E. Hong, S-B., Taghuchi, T., Varga, J. &amp; Samson, R.A. (2007) <em>Aspergillus</em> species identification in the clinical setting. <em>Studies in Mycology</em> 59: 39–46.</p><p> https://doi.org/10.3114/sim.2007.59.05</p><p>Ball, S.L. &amp; Armstrong, K.F. (2006) DNA barcodes for insect pest identification: a test case with tussock moths (Lepidoptera: Lymantriidae). <em>Canadian Journal of Forest Research</em> 36: 337–350.</p><p> https://doi.org/10.1139/x05-276</p><p>Balsamo-Crivelli, G. (1835a) Aufstellung von zwei neuen Arten Mucedineen, <em>Botrytis bassiana</em> und <em>Mucor radicans</em>, etc. <em>Linnaea</em> 10: 609–618.</p><p>Balsamo-Crivelli, G. (1835b) Observazione sopra una nuova spezie dimucedinea del genere <em>Botrytis</em>. <em>Biblioteca Italiana </em>79: 125–129.</p><p>Baum, D.A., Shaw, K.L. (1995) Genealogical perspectives on the species problem. <em>In</em>: Hoch, P.C. &amp; Stephenson, A.G. (Eds.) <em>Experimental and molecular approaches to plant biosystematics</em>. Missouri Botanical Garden, St. Louis, pp. 289–303</p><p>Begerow, D., Nilsson, H., Unterseher, M. &amp; Maier, W. (2010) Current state and perspectives of fungal DNA barcoding and rapid identification procedures. <em>Applied Microbiology and Biotechnology</em> 87: 99–108.</p><p> https://doi.org/10.1007/s00253-010-2585-4</p><p>Benor, S. (2018) Molecular phylogeny of the genus Corchorus (Grewioideae, Malvaceae s.l.) based on nuclear rDNA ITS sequences. <em>The Crop Journal</em> 6: 552–563.</p><p> https://doi.org/10.1016/j.cj.2018.03.007</p><p>Benson, D.A., Clark, K., Karsch-Mizrachi, I., Lipman, D.J., Ostell, J. &amp; Sayers, E.W. (2015) GenBank. <em>Nucleic acids research</em> 43 (D1): 30–35.</p><p> https://doi.org/10.1093/nar/gku1216</p><p>Bischoff, J.F., Rehner, S.A. &amp; Humber, R.A. (2009) A multilocus phylogeny of the <em>Metarhizium</em> <em>anisopliae</em> lineage. <em>Mycologia</em> 101: 512–530.</p><p> https://doi.org/10.3852/07-202</p><p>Bisset, J. &amp; Widden, P. (1988) A new species of <em>Beauveria</em> isolated from Scottish moorland soil. <em>Canadian Journal of Botany</em> 66: 361–362.</p><p> https://doi.org/10.1139/b88-057</p><p>Bortolus, A. (2008) Error cascades in the biological sciences: the unwanted consequences of using bad taxonomy in ecology. <em>Ambio</em>: <em>A journal of the human environment</em> 37: 114–118.</p><p> https://doi.org/10.1579/0044-7447(2008)37[114:ECITBS]2.0.CO;2</p><p>CABI Bioscience (2006) Index Fungorum. Available from: http://www.indexfungorum.org (accessed 20 February 2019)</p><p>Canfora, L., Malusà, E., Tkaczuk, C., Tartanus, M., Łabanowska, B.H. &amp; Pinzari, F. (2016) Development of a method for detection and quantification of <em>B</em>. <em>brongniartii </em>and <em>B</em>. <em>bassiana</em> in soil. <em>Scientific Reports</em> 6: 22933.</p><p> https://doi.org/10.1038/srep22933</p><p>Carlson, A., Justo, A. &amp; Hibbett, D.S. (2014) Species delimitation in Trametes: a comparison of ITS, RPB1, RPB2 and TEF1 gene phylogenies. <em>Mycologia</em> 106: 735–45.</p><p> https://doi.org/10.5248/124.301</p><p>Chen, M.J., Huang, B., Li, Z.Z. &amp; Spatafora, J.W. (2013) Morphological and genetic characterisation of <em>Beauveria sinensis</em> sp. nov. from China. <em>Mycotaxon</em> 124: 301–308.</p><p> https://doi.org/10.5248/124.301</p><p>Chen, W.H., Liu, M., Huang, Z.X., Yang, G.M., Han, Y.F., Liang, J.D. &amp; Liang, Z.Q. (2018) <em>Beauveria majiangensis</em>, a new entomopathogenic fungus from Guizhou, China. <em>Phytotaxa</em> 333: 243–250.</p><p> https://doi.org/10.11646/phytotaxa.333.2.8</p><p>Chen, X., Huang, C., He, L., Zhang, S. &amp; Li, Z. (2015) Molecular tracing of white muscardine in the silkworm, <em>Bombyx mori</em> (Linn.) II. Silkworm white muscardine is not caused by artificial release or natural epizootic of <em>Beauveria bassiana</em> in China. <em>Journal of Invertebrate Pathology</em> 125: 16–22.</p><p> https://doi.org/10.1016/j.jip.2014.12.005</p><p>De Hoog, G.S. (1972) The genera <em>Beauveria</em>, <em>Isaria</em>, <em>Tritirachium</em>, and <em>Acrodontium</em>, gen. nov. <em>Studies in Mycology</em> 1: 1–41.</p><p>De Hoog, G.S. (1978) Notes on fungicolous hyphomycetes and their relatives. <em>Persoonia</em> 10: 33–81.</p><p>De Hoog, G.S. &amp; Rao, V. (1975) Some new Hyphomycetes. <em>Persoonia</em> 8: 207–212.</p><p>Dentinger, B.T.M., Didukh, M.Y. &amp; Moncalvo, J.M. (2011) Comparing COI and ITS as DNA barcode markers for mushrooms and allies (Agaricomycotina). <em>PLoS One</em> 6: e25081.</p><p> https://doi.org/10.1371/journal.pone.0025081</p><p>Dupuis, J.R., Roe, A.D. &amp; Sperling, F.A. (2012) Multi-locus species delimitation in closely related animals and fungi: one marker is not enough. <em>Molecular Ecology</em> 21: 4422–4436.</p><p> https://doi.org/10.1111/j.1365-294X.2012.05642.x</p><p>Garnica, S.M., Weiß, Oertel, B. &amp; Oberwinkler, F. (2005) framework for a phylogenetic classification in the genus <em>Cortinarius</em> (Basidiomycota, Agaricales) derived from morphological and molecular data. <em>Canadian Journal of Botany</em> 83: 1457–1477.</p><p> https://doi.org/10.1139/b05-107</p><p>Garrido-Jurado, I., Fernández-Bravo, M., Campos, C. &amp; Quesada-Moraga, E. (2015) Diversity of entomopathogenic Hypocreales in soil and phylloplanes of five Mediterranean cropping systems. <em>Journal of Invertebrate Pathology </em>130: 97–106.</p><p> https://doi.org/10.1016/j.jip.2015.06.001</p><p>Gazis, R., Rehner, S. &amp; Chaverri, P. (2011) Species delimitation in fungal endophyte diversity studies and its implications in ecological and biogeographic inferences. <em>Molecular Ecology </em>20: 3001–3013.</p><p> https://doi.org/10.1111/j.1365-294X.2011.05110.x</p><p>Hall, T.A. (1999) BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/ 98/NT. <em>Nucleic Acids Symposium</em> 41: 95–98</p><p>Hansen, K. &amp; Olariaga, I. (2015) Species limits and relationships within Otidea inferred from multiple gene phylogenies. <em>Persoonia</em> 35: 148–165.</p><p> https://doi.org/10.3767/003158515X687993</p><p>Harrington, R.C. &amp; Near, T.J. (2012) Phylogenetic and coalescent strategies of species delimitation in Snubnose Darters (Percidae: Etheostoma). <em>Systematic Biology </em>61: 63–79.</p><p> https://doi.org/10.1093/sysbio/syr077</p><p>Harrower, E., Ammirati, J.F., Cappuccino, A.A., Ceska, O., Kranabetter, J.M., Kroeger, P., Lim, S., Taylor, T. &amp; Berbee, M.L. (2011) Cortinarius species diversity in British Columbia and molecular phylogenetic comparison with European specimen sequences. <em>Botany</em> 89: 799–810.</p><p> https://doi.org/10.1139/b11-065</p><p>Hebert, P.D.N., Cywinska, A., Ball, S.L. &amp; deWaard, J.R. (2003) Biological identifications through DNA barcodes. <em>Proceedings of the Royal Society of London</em>. <em>Series B</em>: <em>Biological Sciences</em> 270: 313–21.</p><p> https://doi.org/10.1098/rspb.2002.2218</p><p>Hibbett, D.S., Abarenkov, K., Kõljalg, U., Öpik, M., Chai, B., Cole, J., Wang, Q., Crous, P., Robert, V., Helgason, T., Herr, J.R., Kirk, P., Lueschow, S., O’Donnell, K., Nilsson, R.H., Oono, R., Schoch, C., Smyth, C., Walker, D.M., Porras-Alfaro, A., Taylor, J.W. &amp; Geiser, D.M. (2016) Sequence-based classification and identification of Fungi. <em>Mycologia</em> 108: 1049–1068.</p><p>Hibbett, D.S., Binder, M., Bischoff, J.F., Blackwell, M., Cannon, P.F., Eriksson, O.E., Huhndorf, S., James, T., Kirk, P.M., Lucking, R., Lumbsch, H.T., Lutzoni, F., Matheny, P.B., McLaughlin, D.J., Powell, M.J., Red head, S., Schoch, C.L., Spatafora, J.W., Stalpers, J.A., Vilgalys, R., Aime, M.C., Aptroot, A., Bauer, R., Begerow, D., Benny, G.L., Castlebury, L.A., Crous, P.W., Dai, Y.C., Gams, W., Geiser, D.M., Griffith, G.W., Gueidan, C., Hawksworth, D.L., Hestmark, G., Hosaka, K., Humber, R.A., Hyde, K.D., Ironside, J.E., Koljalg, U., Kurtzman, C.P., Larsson, K.H., Licht wardt, R., Longcore, J., Miadlikowska, J., Miller, A., Moncalvo, J.M., Mozley Standridge, S., Oberwin kler, F., Parmasto, E., Reeb, V., Rogers, J.D., Roux, C., Ryvarden, L., Sampaio, J.P., Schussler, A., Sugiya ma, J., Thorn, R.G., Tibell, L., Untereiner, W.A., Wal ker, C., Wang, Z., Weir, A., Weiss, M., White, M.M., Winka, K., Yao, Y.J. &amp; Zhang, N. (2007) A higher-level phylogenetic classification of the Fungi. <em>Mycological Research</em> 111: 509–547.</p><p> https://doi.org/10.1016/j.mycres.2007.03.004</p><p>Hibbett, D.S., Ohman, A., Glotzer, D., Nuhn, M., Kirk, P. &amp; Nilsson, R.H. (2011) Progress in molecular and morphological taxon discovery in Fungi and options for formal classification of environmental sequences. <em>Fungal Biology Reviews</em> 25: 38e47.</p><p> https://doi.org/10.1016/j.fbr.2011.01.001</p><p>Hickerson, M.J., Meyer, C.P. &amp; Moritz, C. (2006) DNA barcoding will often fail to discover new animal species over broad parameter space. <em>Systematic Biology</em> 55: 729–739.</p><p> https://doi.org/10.1080/10635150600969898</p><p>Hoffman, M.T. &amp; Arnold, A.E. (2008) Geographic locality and host identity shape fungal endophyte communities in cupressaceous trees. <em>Mycological Research</em> 112: 331–344.</p><p> https://doi.org/10.1016/j.mycres.2007.10.014</p><p>Hofstetter, V., Miadlikowska, J., Kauff, F. &amp; Lutzoni, F. (2007) Phylogenetic comparison of protein-coding versus ribosomal RNA-coding sequence data: a case study of the Lecanoromycetes (Ascomycota). <em>Molecular Phylogenetics and Evolution</em> 44: 412–426.</p><p> https://doi.org/10.1016/j.ympev.2006.10.016</p><p>Holder, M. &amp; Lewis, P.O. (2003) Phylogeny estimation: traditional and Bayesian approaches. <em>Nature Reviews Genetics</em> 4, 275–284.</p><p> https://doi.org/10.1038/nrg1044</p><p>Horton, T.R. &amp; Bruns, T.D. (2001) The molecular revolution in ectomycorrhizal ecology: peeking into the black-box. <em>Molecular Ecology</em> 10: 1855–1871.</p><p> https://doi.org/10.1046/j.0962-1083.2001.01333.x</p><p>Katoh, K., Rozewicki, J. &amp; Yamada, K.D. (2017) MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. <em>Briefings in Bioinformatics</em>.</p><p> https://doi.org/10.1093/bib/bbx108</p><p>Imoulan, A., Alaoui, A. &amp; El Meziane, A. (2011) Natural occurrence of soil–borne entomopathogenic fungi in the Moroccan Endemic forest of <em>Argania spinosa</em> and their pathogenicity to <em>Ceratitis capitata</em>. <em>World Journal of Microbiology and Biotechnology</em> 27: 2619–2628.</p><p> https://doi.org/10.1007/s11274-011-0735-1</p><p>Imoulan, A., Hussain, M., Kirk, P.M., El Meziane, A. &amp; Yao, Y.J. (2017) Entomopathogenic fungus <em>Beauveria</em>: Host specificity, ecology and significance of morpho-molecular characterization in accurate taxonomic classification. <em>Journal of Asia</em>-<em>Pacific Entomology</em> 20: 1204–1212.</p><p> https://doi.org/10.1016/j.aspen.2017.08.015</p><p>Imoulan, A., Li, Y., Wang, W.J., El Meziane, A. &amp; Yao, Y.J. (2016a) New record of <em>Beauveria pseudobassiana</em> to Morocco. <em>Mycotaxon</em> 131: 913–923.</p><p> https://doi.org/10.5248/131.913</p><p>Imoulan, A., Wu, H.J., Lu, W.L., Li, Y., Li, B.B., Yang, R.H., Wang, W.J., Wang, X.L., Kirk, P.M. &amp; Yao, Y.J. (2016b) <em>Beauveria medogensis</em> sp. nov., a new fungus of the entomopathogenic genus from China. <em>Journal of Invertebrate Pathology</em> 139: 74–81.</p><p> https://doi.org/10.1016/j.jip.2016.07.006</p><p>Irinyi, L., Lackner, M., De Hoog, G.S. &amp; Meyer, W. (2016) DNA barcoding of fungi causing infections in humans and animals. <em>Fungal Biology</em> 120: 125–136.</p><p> https://doi.org/10.1016/j.funbio.2015.04.007</p><p>Jayasiri, S.C., Hyde, K.D., Ariyawansa, H.A., Bhat, J., Buyck, B., Cai, L., Dai, Y.-C., Abd-Elsalam, K.A., Ertz, D., Hidayat, I., Jeewon, R., Jones, E.B.G., Bahkali, A.H., Karunarathna, S.C., Liu, J.-K., Luangsa-ard, J.J., Lumbsch, H.T., Maharachchikumbura, S.S.N., McKenzie, E.H.C., Moncalvo, J.-M., Ghobad-Nejhad, M., Nilsson, H., Pang, K.-L., Pereira, O.L., Phillips, A.J.L., Raspé, O., Rollins, A.W., Romero, A.I., Etayo, J., Selçuk, F., Stephenson, S.L., Suetrong, S., Taylor, J.E., Tsui, C.K.M., Vizzini, A., Abdel-Wahab, M.A., Wen, T.-C., Boonmee, S., Dai, D.Q., Daranagama, D.A., Dissanayake, A.J., Ekanayaka, A.H., Fryar, S.C., Hongsanan, S., Jayawardena, R.S., Li, W.-J., Perera, R.H., Phookamsak, R., Silva, N.I., Thambugala, K.M., Tian, Q., Wijayawardene, N.N., Zhao, R.-L., Zhao, Q., Kang, J.-C. &amp; Promputtha, I. (2015) The Faces of Fungi database : fungal names linked with morphology, phylogeny and human impacts. <em>Fungal Diversity</em> 74: 3–18.</p><p> https://doi.org/10.1007/s13225-015-0351-8</p><p>Kiss, L. (2012) Limits of nuclear ribosomal DNA internal transcribed spacer (ITS) sequences as species barcodes for Fungi. <em>Proceedings of the National Academy of Sciences</em> 109: E1811.</p><p> https://doi.org/10.1073/pnas.1207143109</p><p>Kõljalg, U., Nilsson, R.H., Abarenkov, K., Tedersoo, L., Taylor, A.F.S., Bahram, M., Bates, ST, Bruns, T.T., Bengtsson–Palme, J., Callaghan, T.M., Douglas, B., Drenkhan, T., Eberhardt, U., Dueñas, M., Grebenc, T., Griffith, G.W., Hartmann, M., Kirk, P.M., Kohout, P., Larsson, E., Lindahl, B.D., Lücking, R., Martín, M.P., Matheny, P.B., Nguyen, N.H., Niskanen, T., Oja, J., Peay, K.G., Peintner, U., Peterson, M., Põldmaa, K., Saag, L., Saar, I., Schüßler, A., Senés, C., Smith, M.E., Suija, A., Taylor, D.E., Telleria, M.T., Weiß, M. &amp; Larsson, K.H. (2013) Towards a unified paradigm for sequence–based identification of fungi. <em>Molecular Ecology</em> 22: 5271–5277.</p><p> https://doi.org/10.1111/mec.12481</p><p>Lacap, D.C., Hyde, K.D. &amp; Liew, E.C.Y. (2003) An evaluation of the fungal ‘morphotype’ concept based on ribosomal DNA sequences. <em>Fungal Diversity</em> 12: 53–66.</p><p>Le Gac, M., Hood, M.E., Fournier, E. &amp; Giraud, T. (2007) Phylogenetic evidence of host-specific cryptic species in the anther smut fungus. <em>Evolution</em> 61: 15–26.</p><p> https://doi.org/10.1111/j.1558-5646.2007.00002.x</p><p>Li, X., Yang, Y., Henry, R.J., Rossetto, M., Wang, Y. &amp; Chen, S. (2015) Plant DNA barcoding: from gene to genome. Biological Reviews 90: 157–166.</p><p> https://doi.org/10.1111/brv.12104</p><p>Lutzoni, F., Kauff, F., Cox, C.J., McLaughlin, D., Celio, G., Dentinger, B., Padamsee, M., Hibbett, D., James, T.Y., Baloch, E., Grube, M., Reeb, V., Hofstetter, V., Schoch, C., Arnold, E., Miadlikowska, J., Spatafora, J., Johnson, D., Hambleton, S., Crockett, M., Shoemaker, R., Sung, G.-H., Lücking, R., Lumbsch, T., O’Donnell, K., Binder, M., Diederich, P., Ertz, D., Gueidan, C., Hansen, K., Harris, R.C., Hosaka, K., Lim, Y.-W., Matheny, B., Nishida, H., Pfister, D., Rogers, J., Rossman, A., Schmitt, I., Sipman, H., Stone, J., Sugiyama, J., Yahr, R. &amp; Vilgalys, R. (2004) Assembling the fungal tree of life: progress, classification and evolution of subcellular traits. <em>American Journal of Botany</em> 91: 1446–1480.</p><p> https://doi.org/10.3732/ajb.91.10.1446</p><p>Mace, G.M. (2004) The role of taxonomy in species conservation. P<em>hilosophical Transactions of the Royal Society B</em> 359: 711–719.</p><p> https://doi.org/10.1098/rstb.2003.1454</p><p>MacLeod, D.M. (1954) Investigation on the genera <em>Beauveria</em> Vuill. and <em>Tritirachium</em> Limber. <em>Canadian Journal of Botany</em> 32: 818–890.</p><p> https://doi.org/10.1139/b54-070</p><p>Marcet-Houben, M. &amp; Gabaldón, T. (2009) The tree versus the forest: the fungal tree of life and the topological diversity within the yeast phylome. <em>PloS one</em> 4: e4357.</p><p> https://doi.org/10.1371/journal.pone.0004357</p><p>Mendes, J., Harris, D.J., Carranza, S. &amp; Salvi, D. (2016) Evaluating the phylogenetic signal limit from mitogenomes, slow evolving nuclear genes, and the concatenation approach. New insights into the Lacertini radiation using fast evolving nuclear genes and species trees. <em>Molecular Phylogenetics and Evolution</em> 100: 254–267.</p><p> https://doi.org/10.1016/j.ympev.2016.04.016</p><p>Meyer, C.P. &amp; Paulay, G. (2005) DNA barcoding: error rates based on comprehensive sampling. <em>PLoS biology </em>3: e422.</p><p> https://doi.org/10.1371/journal.pbio.0030422</p><p>Meyling, N.V. &amp; Eilenberg, J. (2006) Isolation and characterisation of <em>Beauveria bassiana</em> isolates from phylloplanes of hedgerow vegetation. Mycological Research 110: 188–95.</p><p> https://doi.org/10.1016/j.mycres.2005.09.008</p><p>Molnar, I., Gibson, D.M. &amp; Krasnoff, S.B. (2010) Secondary metabolites from entomopathogenic Hypocrealean fungi. <em>Natural Product Reports</em> 27: 1241–1275.</p><p> https://doi.org/10.1039/c001459c</p><p>Nilsson, R.H., Ryberg, M., Abarenkov, K., Sjökvist, E. &amp; Kristiansson, E. (2009) The ITS region as a target for characterization of fungal communities using emerging sequencing technologies. <em>FEMS Microbiology Letters</em> 296: 97–101.</p><p> https://doi.org/10.1111/j.1574-6968.2009.01618.x</p><p>Nilsson, H., Kristiansson, E., Ryberg, M., Hallenbergand, N. &amp; Larsson, K.H. (2008) Intraspecific ITS variability in the kingdom Fungi as expressed in the international sequence databases and it’s implications for molecular species identification. <em>Evolutionary Bioinformatics</em> 4: 193–201.</p><p> https://doi.org/10.4137/EBO.S653</p><p>O’Donnel, K., Cigelnik, E. &amp; Nirenberg, H.I. (1998) Molecular systematics and phylogeography of the <em>Gibberella fujikuroi</em> species complex. <em>Mycologia</em> 90: 465–493.</p><p> https://doi.org/10.2307/3761407</p><p>Ownley, B.H., Griffin, M.R., Klingeman, W.E., Gwinn, K.D., Moulton, J.K. &amp; Pereira, R.M. (2008) <em>Beauveria bassiana</em>: endophytic colonization and plant disease control. <em>Journal of Invertebrate Pathology</em> 98: 267–270.</p><p> https://doi.org/10.1016/j.jip.2008.01.010</p><p>Petch, T. (1926) Studies in entomogenous fungi. <em>Transactions of the British Mycological Society </em>10: 244–271.</p><p> https://doi.org/10.1016/S0007-1536(26)80043-9</p><p>Petit, R.J. &amp; Excoffier, L. (2009) Gene flow and species delimitation. <em>Trends in Ecology &amp; Evolution</em> 24: 386–393.</p><p> https://doi.org/10.1016/j.tree.2009.02.011</p><p>Poczai, P. &amp; Hyvönen, J. (2010) Nuclear ribosomal spacer regions in plant phylogenetics: problems and prospects. <em>Molecular Biology Reports</em> 37: 1897–1912.</p><p> https://doi.org/10.1007/s11033-009-9630-3</p><p>Pringle, A., Baker, D.M., Platt, J.L., Wares, J.P., Latge, J.P. &amp; Taylor, J.W. (2005) Cryptic speciation in the cosmopolitan and clonal human pathogenic fungus <em>Aspergillus fumigatus</em>. <em>Evolution</em> 59: 1886–1899.</p><p> https://doi.org/10.1554/04-241.1</p><p>Rampersad, S.N. (2014) ITS1, 5.8 S and ITS2 secondary structure modelling for intra-specific differentiation among species of the <em>Colletotrichum gloeosporioides sensu lato</em> species complex. <em>SpringerPlus</em> 3: 684.</p><p> https://doi.org/10.1186/2193-1801-3-684</p><p>Rannala, B. (2015) The art and science of species delimitation. <em>Current Zoology</em> 5: 846–853.</p><p> https://doi.org/10.1093/czoolo/61.5.846</p><p>Reeb, V., Lutzoni, F. &amp; Roux, C. (2004) Contribution of RPB2 to multilocus phylogenetic studies of the euascomycetes (Pezizomycotina, Fungi) with special emphasis on the lichen-forming Acarosporaceae and evolution of polyspory. Molecular <em>Phylogenetics and Evolution</em> 32: 1036–1060.</p><p> https://doi.org/10.1016/j.ympev.2004.04.012</p><p>Rehner, S.A, Minnis, A.M., Sung, G.H., Luangsa ard, J.J., Devotto, L. &amp; Humber, R.A. (2011) Phylogeny and systematics of the anamorphic, entomopathogenic genus <em>Beauveria</em>. <em>Mycologia</em> 103: 1055–73.</p><p> https://doi.org/10.3852/10-302</p><p>Rehner, S.A. (2005) Phylogenetics of the insect pathogenic genus <em>Beauveria</em>. <em>In</em>: Vega, F.E. &amp; Blackwell, M. (Eds.) <em>Insect</em>-<em>Fungal Associations</em>: <em>Ecology and Evolution</em>. Oxford University Press, New York, pp. 3–27.</p><p>Rehner, S.A. &amp; Buckley, E.P. (2005) A <em>Beauveria</em> phylogeny inferred from nuclear ITS and <em>EF1</em>–α sequences: evidence for cryptic diversification and links to <em>Cordyceps</em> teleomorphs. <em>Mycologia</em> 97: 84–98.</p><p> https://doi.org/10.1080/15572536.2006.11832842</p><p>Rehner, S.A., Posada, F., Buckley, E.P., Infante, F., Castillo, A. &amp; Vega, F.E. (2006) Phylogenetic origins of African and Neotropical <em>Beauveria bassiana</em> s. l. pathogens of the coffee berry borer, <em>Hypothenemus hampei</em>. <em>Journal of Invertebrate Pathology</em> 93: 11–21.</p><p> https://doi.org/10.3852/14-344</p><p>Robène-Soustrade, I., Jouen, E., Pastou, D., Payet Hoareau, M., Goble, T., Linderme, D., Lefeuvre, P., Calmès, C., Reynaud, B., Nibouche, S. &amp; Costet, L. (2015) Description and phylogenetic placement of <em>Beauveria hoplocheli</em> sp. nov. used in the biological control of the sugarcane white grub, <em>Hoplochelus marginalis</em>, in Reunion Island. <em>Mycologia</em> 107: 1221–1232.</p><p> https://doi.org/10.3852/14-344</p><p>Roe, A.D., Rice, A.V., Bromilow, S.E., Cooke, J.E. &amp; Sperling, F.A. (2010) Multilocus species identification and fungal DNA barcoding: insights from blue stain fungal symbionts of the mountain pine beetle. <em>Molecular Ecology Resources</em> 10: 946–959.</p><p> https://doi.org/10.1111/j.1755-0998.2010.02844.x</p><p>Rosenberg, N.A. (2002) The probability of topological concordance of gene trees and species trees. <em>Theoretical Population Biology</em> 61: 225–247.</p><p> https://doi.org/10.1006/tpbi.2001.1568</p><p>Roy, H.E., Vega, F.E. &amp; Chandler, D. (2010) <em>The ecology of fungal entomopathogens</em>. Springer, The Netherlands, 204 pp.</p><p> https://doi.org/10.1007/978-90-481-3966-8</p><p>Sánchez-Ramírez, S., Tulloss, R.E., Cifuentes-Blanco, J., Guzmán-Dávalos, L., Valenzuela, R., Estrada-Torres, A., Ruán-Soto, F., Díaz-Moreno, R., Hernández-Rico, N., Torres-Gómez, M., León, H. &amp; Moncalvo, J.M. (2015) In and out of refugia: historical patterns of diversity and demography in the North American Caesar’s mushroom species complex. <em>Molecular Ecology</em> 24: 4809–5074.</p><p> https://doi.org/10.1111/mec.13413</p><p>Sanjuan, T., Tabima, J., Restrepo, S., Læssøe, T., Spatafora, J.W. &amp; Franco-Molano, A.E. (2014) Entomopathogens of Amazonian stick insects and locusts are members of the <em>Beauveria</em> species complex (<em>Cordyceps</em> sensu stricto). <em>Mycologia</em> 106: 260–275.</p><p> https://doi.org/10.3852/13-020</p><p>Schilling, M.P., Mullen, S.P., Kronforst, M., Safran, R.J., Nosil, P., Feder, J.L., Gompert, Z. Faxman, S.M. (2018) Transitions from Single-to Multi-Locus Processes during Speciation with Gene Flow. <em>Genes</em> 9 (6): 274.</p><p> https://doi.org/10.3390/genes9060274</p><p>Schneider, S., Rehner, S.A., Widmer, F. &amp; Enkerli, J. (2011) A PCR-based tool for cultivation-independent detection and quantification of <em>Metarhizium</em> clade 1. <em>Journal of Invertebrate Pathology</em> 108: 106–114.</p><p> https://doi.org/10.1016/j.jip.2011.07.005</p><p>Schoch, C.L., Seifert, K.A., Huhndorf, S., Robert, V., Spouge, J.L., Levesque, C.A. &amp; Chen, W. (2012) Nuclear ribosomal internal transcribed spacer (ITS) region as a universal DNA barcode marker for fungi. <em>Proceedings of the National Academy of Sciences USA</em> 109: 6241–6246.</p><p> https://doi.org/10.1073/pnas.1117018109</p><p>Seifert, K.A. (2009) Progress towards DNA barcoding of fungi. <em>Molecular Ecology Resources</em> 9:83–9.</p><p> https://doi.org/10.1111/j.1755-0998.2009.02635.x</p><p>Simmons, D.R., Lund, J., Levitsky, T. &amp; Groden, E. (2015) Ophiocordyceps myrmicarum, a new species infecting invasive Myrmica rubra in Maine. <em>Journal of Invertebrate Pathology</em> 125: 23–30.</p><p> https://doi.org/10.1016/j.jip.2014.12.010</p><p>Smith, M.E., Douhan, G.W. &amp; Rizzo, D.M. (2007) Intra-specific and intra- sporocarp ITS variation of ectomycorrhizal fungi as assessed by rDNA sequencing of sporocarps and pooled ectomycorrhizal roots from a Quercus woodland. <em>Mycorrhiza</em> 18: 15–22.</p><p> https://doi.org/10.1007/s00572-007-0148-z</p><p>Steen, F., López-bautista, J.M., Zuccarello, G.C. &amp; Clerck, O.D.E. (2014) DNA-based species delimitation in algae. <em>European Journal of Phycology</em> 49: 179–196.</p><p> https://doi.org/10.1080/09670262.2014.904524</p><p>Steinhaus, E.A. (1949) <em>Principles of Insect Pathology</em>. Hafner Publishing Company, New York.</p><p>Sung, G.H., Hywel-Jones, N.L., Sung, J.M., Luangsa-Ard, J.J., Shrestha, B. &amp; Spatafora, J.W. (2007) Phylogenetic classification of <em>Cordyceps</em> and the clavicipitaceous fungi. <em>Studies in Mycology </em>57: 5–59.</p><p> https://doi.org/10.3114/sim.2007.57.01</p><p>Swofford DL. (2003) <em>PAUP*: phylogenetic analysis using parsimony (*and other methods).</em> Version 4. Sunderland, Massachusetts: Sinauer Associates.</p><p>Tamura, K., Stecher, G., Peterson, D., Filipski, A. &amp; Kumar, S. (2013) MEGA 6: Molecular evolutionary genetics analysis. <em>Molecular Biology and Evolution</em> 30: 2725–2729.</p><p> https://doi.org/10.1093/molbev/mst197</p><p>Tang, C.Q., Humphreys, A.M., Fontaneto, D. &amp; Barraclough, T.G. (2014) Effects of phylogenetic reconstruction method on the robustness of species delimitation using single-locus data. <em>Methods in Ecology and Evolution</em> 5: 1086–1094.</p><p> https://doi.org/10.1111/2041-210X.12246</p><p>Taylor, J.W., Jacobson, D.J., Kroken, S., Kasuga, T., Geiser, D.M., Hibbett, D.S. &amp; Fisher, M.C. (2000) Phylogenetic species recognition and species concepts in Fungi. <em>Fungal Genetics and Biology</em> 31: 21–32.</p><p> https://doi.org/10.1006/fgbi.2000.1228</p><p>Teasdale, S.E., Beulke, A.K., Guy, P.L. &amp; Orlovich, D.A. (2013) Environmental barcoding of the ectomycorrhizal fungal genus Cortinarius. <em>Fungal Diversity</em> 58: 299–310.</p><p> https://doi.org/10.1007/s13225-012-0218-1</p><p>Teletchea, F. (2010) After 7 years and 1000 citations: comparative assessment of the DNA barcoding and the DNA taxonomy proposals for taxonomists and non-taxonomists. <em>Mitochondrion</em> <em>DNA</em> 21: 206–226.</p><p> https://doi.org/10.3109/19401736.2010.532212</p><p>Vaidya, G., Lohman, D. &amp; Meier, R. (2011) SequenceMatrix: concatenation software for the fast assembly of multi-gene datasets with character set and codon information. <em>Cladistics</em> 27: 171–180.</p><p> https://doi.org/10.1111/j.1096-0031.2010.00329.x</p><p>Vilgalys, R. &amp; Gonzalez, D. (1990) Organization of ribosomal DNA in the basidiomycete <em>Thanatephorus praticola</em>. <em>Current Genetics</em> 18 : 277–280.</p><p> https://doi.org/10.1007/BF00318394</p><p>Vuillemin, P. (1912) <em>Beauveria</em>- nouveau genre de Verticilliacies. <em>Bulletin de la Société Botanique de France</em> 59: 34–40.</p><p> https://doi.org/10.1080/00378941.1912.10832379</p><p>Wang, S.B., Miao, X.X., Zhao, W.G., Huang, B., Fan, M.Z., Li, Z.Z. &amp; Huang, Y.P. (2005) Genetic diversity and population structure among strains of the entomopathogenic fungus, <em>Beauveria bassiana</em>, as revealed by inter-simple sequence repeats (ISSR). <em>Mycological Research</em> 109: 1364–1372.</p><p> https://doi.org/10.1017/S0953756205003709</p><p>Wang, Y., Li, T., Li, Y., Björn, L.O., Rosendahl, S., Olsson, P.A., Li, S. &amp; Fu, X. (2015) Community dynamics of arbuscular mycorrhizal fungi in high-input and intensively irrigated rice cultivation systems. <em>Applied and Environmental Microbiology</em> 81: 2958–65.</p><p> https://doi.org/10.1128/AEM.03769-14</p><p>White, T.J., Bruns, T., Lee, S. &amp; Taylor, J. (1990) Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics. <em>In</em>: <em>PCR protocols</em>: <em>a guide to methods and applications</em>. Academic Press, California, pp. 315–322.</p><p> https://doi.org/10.1016/B978-0-12-372180-8.50042-1</p><p>Wiens, J.J. (2007) Species delimitation: new approaches for discovering diversity. <em>Systematic Biology</em> 56: 875–878.</p><p> https://doi.org/10.1080/10635150701748506</p><p>Xu, Y., Zhan, J., Wijeratne, E.M., Burns, A.M., Gunatilaka, A.A. &amp; Molnar, I. (2007) Cytotoxic and antihaptotactic beauvericin analogues from precursor-directed biosynthesis with the insect pathogen <em>Beauveria bassiana</em> ATCC 7159. <em>Journal of Natural Products </em>70: 1467–1471.</p><p> https://doi.org/10.1021/np070262f</p><p>Yang, Z. &amp; Rannala, B. (2010) Bayesian species delimitation using multilocus sequence data. <em>Proceedings of the National Academy of Sciences</em> 107: 9264–9269.</p><p> https://doi.org/10.1073/pnas.0913022107</p><p>Yao, J., Zhang, L.W., Chen, X.J. &amp; Li, Z.Z. (2012) A population survey of <em>Beauveria bassiana</em> in the microhabitat of the red turpentine bark beetle, <em>Dendroctonus valens</em>, in Chinese pine forests. <em>Biocontrol Science and Technology</em> 22: 1254–1267.</p><p> https://doi.org/10.1080/09583157.2012.724382</p><p>Zhang, S.L., He, L.M., Chen, X. &amp; Hueng, B. (2012) <em>Beauveria lii</em> sp. nov. isolated from <em>Henosepilachna vigintioctopunctata</em>. <em>Mycotaxon</em> 121: 199–206.</p><p> https://doi.org/10.5248/121.199</p><p>Zinger, L. &amp; Philippe, H. (2016) Coalescing molecular evolution and DNA barcoding. <em>Molecular Ecology</em> 25: 1908–1910.</p><p> https://doi.org/10.1111/mec.13639</p></p>
					</div>
		<br />
		</div>
	
							<br />
		<br />
		DOI: <a id="pub-id::doi" href="https://doi.org/10.11646/phytotaxa.413.4.1">https://doi.org/10.11646/phytotaxa.413.4.1</a>																				

<div class="separator"></div>

<h3>Refbacks</h3>

<ul>
				<li>There are currently no refbacks.</li>
	</ul>


		
<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=https%3A%2F%2Fwww.biotaxa.org%2FPhytotaxa%2Farticle%2Fview%2F46643&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.title=Phytotaxa&amp;rft.jtitle=Phytotaxa&amp;rft.atitle=%3Cp%3E%3Cstrong%3ETransitions+from+Single-+to+Multi-Locus+approach+in+determining+cryptic+and+describing+new+species+within+%3Cem%3EBeauveria%3C%2Fem%3E+genus+%28Cordycipitaceae%2C+Hypocreales%29%3A+a+review%3C%2Fstrong%3E%3C%2Fp%3E&amp;rft.artnum=phytotaxa.413.4.1&amp;rft.stitle=Phytotaxa&amp;rft.volume=413&amp;rft.issue=4&amp;rft.aulast=ABDESSAMAD&amp;rft.aufirst=IMOULAN&amp;rft.auinit=&amp;rft.date=2019-08-05&amp;rft.au=IMOULAN+ABDESSAMAD&amp;rft_id=info%3Adoi%2F10.11646%2Fphytotaxa.413.4.1&amp;rft.pages=257%E2%80%93273&amp;rft.issn=1179-3155&amp;rft.eissn=1179-3163"></span>
<!-- Google Analytics -->

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-89286081-1', 'auto');
ga('send', 'pageview');

</script>

<!-- End Google Analytics -->
<br /><br />
<p><span style="color: #0099cc; font-family: 'Times New Roman'; font-size: small;"><img src="/public/site/images/mapress/mpicon.gif" alt="" /></span></p><p><span style="color: #0099cc; font-family: 'Times New Roman'; font-size: small;">ISSN <span lang="EN-AU">1179-3155 (print); </span></span><span style="color: #0099cc; font-family: 'Times New Roman'; font-size: small;">ISSN 1179-3163 (online)</span></p><p>Published by <a title="Magnolia Press" href="http://www.mapress.com/phytotaxa/index.htm" target="_blank">Magnolia Press</a>, Auckland, New Zealand</p><div> </div>
</div><!-- content -->
</div><!-- main -->
</div><!-- body -->

<script type="text/javascript">

<!--
	// Open "Define Terms" context when double-clicking any text
	function openSearchTermWindow(url) {
		var term;
		if (window.getSelection) {
			term = window.getSelection();
		} else if (document.getSelection) {
			term = document.getSelection();
		} else if(document.selection && document.selection.createRange && document.selection.type.toLowerCase() == 'text') {
			var range = document.selection.createRange();
			term = range.text;
		}
		if (term != ""){
			if (url.indexOf('?') > -1) openRTWindowWithToolbar(url + '&defineTerm=' + term);
			else openRTWindowWithToolbar(url + '?defineTerm=' + term);
		}
	}

	if(document.captureEvents) {
		document.captureEvents(Event.DBLCLICK);
	}

	// Make sure to only open the reading tools when double clicking within the galley
	if (document.getElementById('inlinePdfResizer')) {
		context = document.getElementById('inlinePdfResizer');
	}
	else if (document.getElementById('content')) {
		context = document.getElementById('content');
	}
	else {
		context = document;
	}

	context.ondblclick = new Function("openSearchTermWindow('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.413.4.1/0/446')");
// -->

</script>


</div> <!-- container -->
<script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam-cell.nr-data.net","licenseKey":"4f72ad4ec1","applicationID":"416471445","transactionName":"ZF0AZkZRWRVUVEINDF0XN0BdH14IUVJOShNbSA==","queueTime":0,"applicationTime":299,"atts":"SBoDEA5LShs=","errorBeacon":"bam-cell.nr-data.net","agent":""}</script></body>
</html>