<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
	<title>   Marileptolyngbya sina     gen. nov., sp. nov.  and  Salileptolyngbya diazotrophicum gen. nov., sp. nov.  (Synechococcales, Cyanobacteria), species of cyanobacteria isolated from a marine ecosystem   | ZHOU | Phytotaxa</title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"VgMHUldUABAEVVdWAAADUlE=",licenseKey:"4f72ad4ec1",applicationID:"416471445"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var i=e[n]={exports:{}};t[n][0].call(i.exports,function(e){var i=t[n][1][e];return r(i||e)},i,i.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var i=0;i<n.length;i++)r(n[i]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var i,o=t("ee"),a=t(23),c={};try{i=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,i.indexOf("dev")!==-1&&(c.dev=!0),i.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&o.on("internal-error",function(t){r(t.stack)}),c.dev&&o.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{p?p-=1:i(c||new UncaughtException(t,e,n),!0)}catch(f){try{o("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function i(t,e){var n=e?null:s.now();o("err",[t,n])}var o=t("handle"),a=t(24),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,l="nr@seenError",p=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(9),t(8),"addEventListener"in window&&t(5),s.xhrWrappable&&t(10),d=!0)}c.on("fn-start",function(t,e,n){d&&(p+=1)}),c.on("fn-err",function(t,e,n){d&&!n[l]&&(f(n,l,function(){return!0}),this.thrown=!0,i(n))}),c.on("fn-end",function(){d&&!this.thrown&&p>0&&(p-=1)}),c.on("internal-error",function(t){o("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var i=t("ee"),o=t("handle"),a=t(9),c=t(8),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",l="resource",p="-start",h="-end",m="fn"+p,w="fn"+h,v="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(7),"addEventListener"in window&&t(5);var x=NREUM.o.EV;i.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),i.on(w,function(t,e){var n=t[0];n instanceof x&&o("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(w,function(t,e){o(v,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(w,function(t,e){o(v,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),i.on(g+p,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),i.on(g+h,function(t){o("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){o(d,[window.performance.getEntriesByType(l)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){o(d,[window.performance.getEntriesByType(l)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],5:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&i(e)}function i(t){c.inPlace(t,[u,d],"-",o)}function o(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(i(window),i(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],6:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=o(arguments),e={};i.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return i.emit(n+"start",[t,a],c),c.then(function(t){return i.emit(n+"end",[null,t],c),t},function(t){throw i.emit(n+"end",[t],c),t})})}var i=t("ee").get("fetch"),o=t(24),a=t(23);e.exports=i;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,l=c.Response,p=c.fetch,h="prototype",m="nr@context";d&&l&&p&&(a(u,function(t,e){r(d[h],e,f),r(l[h],e,f)}),r(c,"fetch",s),i.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),i.emit(s+"done",[null,e],n)}else i.emit(s+"done",[t],n)}))},{}],7:[function(t,e,n){var r=t("ee").get("history"),i=t("wrap-function")(r);e.exports=r;var o=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;o&&o.pushState&&o.replaceState&&(a=o),i.inPlace(a,["pushState","replaceState"],"-")},{}],8:[function(t,e,n){var r=t("ee").get("raf"),i=t("wrap-function")(r),o="equestAnimationFrame";e.exports=r,i.inPlace(window,["r"+o,"mozR"+o,"webkitR"+o,"msR"+o],"raf-"),r.on("raf-start",function(t){t[0]=i(t[0],"fn-")})},{}],9:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function i(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var o=t("ee").get("timer"),a=t("wrap-function")(o),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=o,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),o.on(s+u,r),o.on(c+u,i)},{}],10:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function i(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function o(t){y.push(t),h&&(b?b.then(a):w?w(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(5);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),l=NREUM.o,p=l.XHR,h=l.MO,m=l.PR,w=l.SI,v="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new p(t);try{u.emit("new-xhr",[e],e),e.addEventListener(v,i,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(p,x),x.prototype=p.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),o(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!w&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===v||a()})},{}],11:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=p.generateSpanId(),m=p.generateTraceId(),w=Date.now(),v={spanId:h,traceId:m,timestamp:w};return(t.sameOrigin||s(t)&&l())&&(v.traceContextParentHeader=i(h,m),v.traceContextStateHeader=o(h,w,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(v.newrelicHeader=a(h,m,w,n,r,f)),v}function i(t,e){return"00-"+e+"-"+t+"-01"}function o(t,e,n,r,i){var o=0,a="",c=1,s="",f="";return i+"@nr="+o+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,i,o){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:i,id:t,tr:e,ti:n}};return o&&r!==o&&(c.d.tk=o),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var i=h(n.allowed_origins[r]);if(t.hostname===i.hostname&&t.protocol===i.protocol&&t.port===i.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function l(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var p=t(20),h=t(13);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],12:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<l;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):o(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function i(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function o(t,e){t.params.status=e.status;var n=w(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(13),f=t(11).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],l=d.length,p=t("id"),h=t(17),m=t(16),w=t(14),v=window.XMLHttpRequest;a.features.xhr=!0,t(10),t(6),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){o(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],i=this;if(n&&r){var o=m(r);o&&(n.txSize=o)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||i.loadCaptureCalled||(i.params.aborted=!0),("load"!==t.type||i.called===i.totalCbs&&(i.onloadCalled||"function"!=typeof e.onload))&&i.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<l;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof v&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof v&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof v&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var i=f(this.parsedOrigin);if(i&&(i.newrelicHeader||i.traceContextParentHeader))if("string"==typeof t[0]){var o={};for(var a in r)o[a]=r[a];o.headers=new Headers(r.headers||{}),e(o.headers,i)&&(this.dt=i),t.length>1?t[1]=o:t.push(o)}else t[0]&&t[0].headers&&e(t[0].headers,i)&&(this.dt=i)})}},{}],13:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,i={};e.href=t,i.port=e.port;var o=e.href.split("://");!i.port&&o[1]&&(i.port=o[1].split("/")[0].split("@").pop().split(":")[1]),i.port&&"0"!==i.port||(i.port="https"===o[0]?"443":"80"),i.hostname=e.hostname||n.hostname,i.pathname=e.pathname,i.protocol=o[0],"/"!==i.pathname.charAt(0)&&(i.pathname="/"+i.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return i.sameOrigin=a&&(!e.hostname||c),"/"===i.pathname&&(r[t]=i),i}},{}],14:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?i(t.response):"text"===n||""===n||void 0===n?i(t.responseText):void 0}var i=t(16);e.exports=r},{}],15:[function(t,e,n){function r(){}function i(t,e,n){return function(){return o(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var o=t("handle"),a=t(23),c=t(24),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",p=l+"ixn-";a(d,function(t,e){u[e]=i(l+e,!0,"api")}),u.addPageAction=i(l+"addPageAction",!0),u.setCurrentRouteName=i(l+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,i="function"==typeof e;return o(p+"tracer",[f.now(),t,n],r),function(){if(s.emit((i?"":"no-")+"fn-start",[f.now(),r,i],n),i)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=i(p+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),o("err",[t,f.now(),!1,e])}},{}],16:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],17:[function(t,e,n){var r=0,i=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);i&&(r=+i[1]),e.exports=r},{}],18:[function(t,e,n){function r(){return c.exists&&performance.now?Math.round(performance.now()):(o=Math.max((new Date).getTime(),o))-a}function i(){return o}var o=(new Date).getTime(),a=o,c=t(25);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=i},{}],19:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function i(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function o(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!w){var e=Math.round(t.timeStamp),n={type:t.type};e<=l.now()?n.fid=l.now()-e:e>l.offset&&e<=Date.now()?(e-=l.offset,n.fid=l.now()-e):e=l.now(),w=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[l.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),l=t("loader"),p=t(22),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(i);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(o);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var w=!1,v=["click","keydown","mousedown","pointerdown","touchstart"];v.forEach(function(t){document.addEventListener(t,a,!1)})}p(c)}},{}],20:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var i,o="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<o.length;c++)i=o[c],"x"===i?a+=t().toString(16):"y"===i?(i=3&t()|8,a+=i.toString(16)):a+=i;return a}function i(){return a(16)}function o(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,i=window.crypto||window.msCrypto;i&&i.getRandomValues&&Uint8Array&&(n=i.getRandomValues(new Uint8Array(31)));for(var o=[],a=0;a<t;a++)o.push(e().toString(16));return o.join("")}e.exports={generateUuid:r,generateSpanId:i,generateTraceId:o}},{}],21:[function(t,e,n){function r(t,e){if(!i)return!1;if(t!==i)return!1;if(!e)return!0;if(!o)return!1;for(var n=o.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var i=null,o=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(i="Safari",o=s[1])}e.exports={agent:i,version:o,match:r}},{}],22:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[i]?"hidden":"visible")}"addEventListener"in document&&o&&document.addEventListener(o,e,!1)}e.exports=r;var i,o,a;"undefined"!=typeof document.hidden?(i="hidden",o="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",o="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",o="webkitvisibilitychange",a="webkitVisibilityState")},{}],23:[function(t,e,n){function r(t,e){var n=[],r="",o=0;for(r in t)i.call(t,r)&&(n[o]=e(r,t[r]),o+=1);return n}var i=Object.prototype.hasOwnProperty;e.exports=r},{}],24:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,i=n-e||0,o=Array(i<0?0:i);++r<i;)o[r]=t[e+r];return o}e.exports=r},{}],25:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function i(t){function e(t){return t&&t instanceof r?t:t?s(t,c,o):o()}function n(n,r,i,o){if(!l.aborted||o){t&&t(n,r,i);for(var a=e(i),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function p(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function w(t){return d[t]=d[t]||i(n)}function v(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:p,addEventListener:p,removeEventListener:h,emit:n,get:w,listeners:m,context:e,buffer:v,abort:a,aborted:!1};return x}function o(){return new r}function a(){(u.api||u.feature)&&(l.aborted=!0,u=l.backlog={})}var c="nr@context",s=t("gos"),f=t(23),u={},d={},l=e.exports=i();l.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(i.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(o){}return t[e]=r,r}var i=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){i.buffer([t],r),i.emit(t,e,n)}var i=t("ee").get("handle");e.exports=r,r.ee=i},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,o,function(){return i++})}var i=1,o="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!b++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();s(g,function(e,n){t[e]||(t[e]=n)});var n=a();c("mark",["onload",n+x.offset],null,"api"),c("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function i(){"complete"===l.readyState&&o()}function o(){c("mark",["domContent",a()+x.offset],null,"api")}var a=t(18),c=t("handle"),s=t(23),f=t("ee"),u=t(21),d=window,l=d.document,p="addEventListener",h="attachEvent",m=d.XMLHttpRequest,w=m&&m.prototype;NREUM.o={ST:setTimeout,SI:d.setImmediate,CT:clearTimeout,XHR:m,REQ:d.Request,EV:d.Event,PR:d.Promise,MO:d.MutationObserver};var v=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1184.min.js"},y=m&&w&&w[p]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:a.getLastTimestamp(),now:a,origin:v,features:{},xhrWrappable:y,userAgent:u};t(15),t(19),l[p]?(l[p]("DOMContentLoaded",o,!1),d[p]("load",r,!1)):(l[h]("onreadystatechange",i),d[h]("onload",r)),c("mark",["firstbyte",a.getLastTimestamp()],null,"api");var b=0},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var i=t("ee"),o=t(24),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,i){function nrWrapper(){var r,a,c,s;try{a=this,r=o(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){l([f,"",[r,a,i],c])}u(e+"start",[r,a,i],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,i,o){i||(i="");var a,c,s,f="-"===i.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+i:i,o,c))}function u(n,r,i){if(!s||e){var o=s;s=!0;try{t.emit(n,r,i,e)}catch(a){l([a,n,r,i])}s=o}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){l([r])}for(var i in t)c.call(t,i)&&(e[i]=t[i]);return e}function l(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=i),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,12,4,3]);</script>
	<meta name="description" content="   Marileptolyngbya sina     gen. nov., sp. nov.  and  Salileptolyngbya diazotrophicum gen. nov., sp. nov.  (Synechococcales, Cyanobacteria), species of cyanobacteria isolated from a marine ecosystem  " />
			<meta name="keywords" content="cyanobacteria, ecology, internal transcribed spacer, morphology, nifH, Algae" />
	
	
	<link rel="schema.DC" href="http://purl.org/dc/elements/1.1/" />

	<meta name="DC.Contributor.Sponsor" xml:lang="en" content=""/>
	<meta name="DC.Creator.PersonalName" content="WEI-GUO ZHOU"/>
	<meta name="DC.Creator.PersonalName" content="DE-WEN DING"/>
	<meta name="DC.Creator.PersonalName" content="QING-SONG YANG"/>
	<meta name="DC.Creator.PersonalName" content="MANZOOR AHMAD"/>
	<meta name="DC.Creator.PersonalName" content="YUAN-ZHOU ZHANG"/>
	<meta name="DC.Creator.PersonalName" content="XIAN-CHENG LIN"/>
	<meta name="DC.Creator.PersonalName" content="YAN-YING ZHANG"/>
	<meta name="DC.Creator.PersonalName" content="JUAN LING"/>
	<meta name="DC.Creator.PersonalName" content="JUN-DE DONG"/>
	<meta name="DC.Date.created" scheme="ISO8601" content="2018-12-13"/>
	<meta name="DC.Date.dateSubmitted" scheme="ISO8601" content="2018-03-02"/>
	<meta name="DC.Date.issued" scheme="ISO8601" content="2018-12-13"/>
	<meta name="DC.Date.modified" scheme="ISO8601" content="2018-12-13"/>
	<meta name="DC.Description" xml:lang="en" content=" We isolated two filamentous, non-heterocystous cyanobacteria, namely SCSIO T-2 from Xincun Bay, Hainan Island and SCSIO 43686 from the northern South China Sea, China. The taxonomic positions of the studied strains were determined by a polyphasic approach that used morphological, molecular, and ecological analyses. The two cyanobacterial strains presented different morphology regarding their cell size and sheaths. Sequencing of the 16S rRNA gene of SCSIO T-2 showed 95% similarity with  Leptolyngbya  sp. CENA 134 (HQ730083) and SCSIO 43686 displayed 97% similarity with  Leptolyngbya  sp. L21-BG-2 (KJ206339). In addition, the  nifH  gene sequence analysis of SCSIO T-2 showed 91% similarity with  Lyngbya lagerheimii  UTEX 1930 (L15550), while SCSIO 43686 had an 88% similarity with  Leptolyngbya minuta  LEGE 07181 (KC256775). Both strains’ secondary structures of D1-D1’, Box-B, and V3 regions within the 16S–23S internal transcribed spacer (ITS) region were quite distinct from their closely related cyanobacterial strains. In addition, compared with SCSIO 43686, the SCSIO T-2 strain displayed relatively high light utilization yet low sensitivity to light intensity. Based on these characteristics, the name  Marileptolyngbya sina  SCSIO T-2 is described, and a new genus named  Salileptolyngbya  is erected to encompass a monophyletic group. We propose  Salileptolyngbya diazotrophicum  SCSIO 43686 for the type species of this new genus. "/>
	<meta name="DC.Format" scheme="IMT" content="application/zip"/>
	<meta name="DC.Format" scheme="IMT" content="application/pdf"/>
	<meta name="DC.Identifier" content="phytotaxa.383.1.4"/>
	<meta name="DC.Identifier.pageNumber" content="75–92"/>
							<meta name="DC.Identifier.DOI" content="10.11646/phytotaxa.383.1.4"/>
		<meta name="DC.Identifier.URI" content="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.383.1.4"/>
	<meta name="DC.Language" scheme="ISO639-1" content="en"/>
	<meta name="DC.Rights" content="Copyright (c) 2018 Phytotaxa" />
	<meta name="DC.Rights" content=""/>
	<meta name="DC.Source" content="Phytotaxa"/>
	<meta name="DC.Source.ISSN" content="1179-3163"/>
	<meta name="DC.Source.Issue" content="1"/>	<meta name="DC.Source.URI" content="https://www.biotaxa.org/Phytotaxa"/>
	<meta name="DC.Source.Volume" content="383"/>						<meta name="DC.Subject" xml:lang="en" content="cyanobacteria, ecology, internal transcribed spacer, morphology, nifH, Algae"/>
				<meta name="DC.Title" content="   Marileptolyngbya sina     gen. nov., sp. nov.  and  Salileptolyngbya diazotrophicum gen. nov., sp. nov.  (Synechococcales, Cyanobacteria), species of cyanobacteria isolated from a marine ecosystem  "/>
		<meta name="DC.Type" content="Text.Serial.Journal"/>
	<meta name="DC.Type.articleType" content="Article"/>
		<meta name="gs_meta_revision" content="1.1" />
	<meta name="citation_journal_title" content="Phytotaxa"/>
	<meta name="citation_issn" content="1179-3163"/>
        <meta name="citation_author" content="WEI-GUO ZHOU"/>
        <meta name="citation_author_institution" content="CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China"/>
        <meta name="citation_author" content="DE-WEN DING"/>
        <meta name="citation_author_institution" content="CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
The First Institute of Oceanography, State Oceanic Administration (SOA), Qingdao, 266061, China"/>
        <meta name="citation_author" content="QING-SONG YANG"/>
        <meta name="citation_author_institution" content="CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China"/>
        <meta name="citation_author" content="MANZOOR AHMAD"/>
        <meta name="citation_author_institution" content="CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China"/>
        <meta name="citation_author" content="YUAN-ZHOU ZHANG"/>
        <meta name="citation_author_institution" content="CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China"/>
        <meta name="citation_author" content="XIAN-CHENG LIN"/>
        <meta name="citation_author_institution" content="CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China"/>
        <meta name="citation_author" content="YAN-YING ZHANG"/>
        <meta name="citation_author_institution" content="CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
Tropical Marine Biological Research Station in Hainan,Chinese Academy of Sciences, Sanya, 572000, China"/>
        <meta name="citation_author" content="JUAN LING"/>
        <meta name="citation_author_institution" content="CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China"/>
        <meta name="citation_author" content="JUN-DE DONG"/>
        <meta name="citation_author_institution" content="CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
Tropical Marine Biological Research Station in Hainan,Chinese Academy of Sciences, Sanya, 572000, China"/>
<meta name="citation_title" content="   Marileptolyngbya sina     gen. nov., sp. nov.  and  Salileptolyngbya diazotrophicum gen. nov., sp. nov.  (Synechococcales, Cyanobacteria), species of cyanobacteria isolated from a marine ecosystem  "/>

	<meta name="citation_date" content="2018/12/13"/>

	<meta name="citation_volume" content="383"/>
	<meta name="citation_issue" content="1"/>

			<meta name="citation_firstpage" content="75"/>
				<meta name="citation_lastpage" content="92"/>
								<meta name="citation_doi" content="10.11646/phytotaxa.383.1.4"/>
		<meta name="citation_abstract_html_url" content="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.383.1.4"/>
	<meta name="citation_language" content="en"/>
						<meta name="citation_keywords" xml:lang="en" content="cyanobacteria, ecology, internal transcribed spacer, morphology, nifH, Algae"/>
									<meta name="citation_fulltext_html_url" content="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.383.1.4/37618"/>
								<meta name="citation_pdf_url" content="https://www.biotaxa.org/Phytotaxa/article/download/phytotaxa.383.1.4/37638"/>
				

	<link rel="stylesheet" href="https://www.biotaxa.org/lib/pkp/styles/pkp.css" type="text/css" />
	<link rel="stylesheet" href="https://www.biotaxa.org/lib/pkp/styles/common.css" type="text/css" />
	<link rel="stylesheet" href="https://www.biotaxa.org/styles/common.css" type="text/css" />
	<link rel="stylesheet" href="https://www.biotaxa.org/styles/compiled.css" type="text/css" />
	<link rel="stylesheet" href="https://www.biotaxa.org/styles/articleView.css" type="text/css" />
			<link rel="stylesheet" href="https://www.biotaxa.org/lib/pkp/styles/rtEmbedded.css" type="text/css" />
	
	
	
	<link rel="stylesheet" href="https://www.biotaxa.org/styles/sidebar.css" type="text/css" />		<link rel="stylesheet" href="https://www.biotaxa.org/styles/rightSidebar.css" type="text/css" />	
			<link rel="stylesheet" href="https://www.biotaxa.org/public/site/sitestyle.css" type="text/css" />
			<link rel="stylesheet" href="https://www.biotaxa.org/plugins/blocks/languageToggle/styles/languageToggle.css" type="text/css" />
	
	<!-- Base Jquery -->
	<script type="text/javascript" src="//www.google.com/jsapi"></script>
	<script type="text/javascript">
		// Provide a local fallback if the CDN cannot be reached
		if (typeof google == 'undefined') {
			document.write(unescape("%3Cscript src='https://www.biotaxa.org/lib/pkp/js/lib/jquery/jquery.min.js' type='text/javascript'%3E%3C/script%3E"));
			document.write(unescape("%3Cscript src='https://www.biotaxa.org/lib/pkp/js/lib/jquery/plugins/jqueryUi.min.js' type='text/javascript'%3E%3C/script%3E"));
		} else {
			google.load("jquery", "1.4.4");
			google.load("jqueryui", "1.8.6");
		}
	</script>
	
	<!-- Compiled scripts -->
			
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/lib/jquery/plugins/jquery.tag-it.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/lib/jquery/plugins/jquery.cookie.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/functions/fontController.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/functions/general.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/functions/jqueryValidatorI18n.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/Helper.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/ObjectProxy.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/Handler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/linkAction/LinkActionRequest.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/classes/features/Feature.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/SiteHandler.js"></script><!-- Included only for namespace definition -->
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/UrlInDivHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/AutocompleteHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/ExtrasOnDemandHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/form/FormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/form/AjaxFormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/form/ClientFormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/grid/GridHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/controllers/linkAction/LinkActionHandler.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/js/pages/search/SearchFormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/js/statistics/ReportGeneratorFormHandler.js"></script>
<script type="text/javascript" src="https://www.biotaxa.org/plugins/generic/lucene/js/LuceneAutocompleteHandler.js"></script>

<script type="text/javascript" src="https://www.biotaxa.org/lib/pkp/js/lib/jquery/plugins/jquery.pkp.js"></script>	
	
	<script type="text/javascript" src="https://www.biotaxa.org/js/relatedItems.js"></script>
	<script type="text/javascript" src="https://www.biotaxa.org/js/inlinePdf.js"></script>
	<script type="text/javascript" src="https://www.biotaxa.org/js/pdfobject.js"></script>

</head>
<body id="pkp-common-openJournalSystems">

<div id="container">

<div id="header">
<div id="headerTitle">
<h1>
	<img src="https://www.biotaxa.org/public/journals/2/pageHeaderTitleImage_en_US.jpg" width="1101" height="98" alt="Page Header" />
</h1>
</div>
</div>

<div id="body">

	<div id="sidebar">
							<div id="rightSidebar">
				<div class="block" id="sidebarDevelopedBy">
	<a class="blockTitle" href="http://pkp.sfu.ca/ojs/" id="developedBy">Open Journal Systems</a>
</div><div class="block" id="sidebarHelp">
	<a class="blockTitle" href="javascript:openHelp('https://www.biotaxa.org/Phytotaxa/help')">Journal Help</a>
</div><div class="block" id="sidebarSubscription">
	<span class="blockTitle">Subscription</span>
							Login to verify subscription
			</div><div class="block" id="sidebarUser">
			<span class="blockTitle">User</span>
	
												<form method="post" action="https://www.biotaxa.org/Phytotaxa/login/signIn">
					<table>
						<tr>
							<td><label for="sidebar-username">Username</label></td>
							<td><input type="text" id="sidebar-username" name="username" value="" size="12" maxlength="32" class="textField" /></td>
						</tr>
						<tr>
							<td><label for="sidebar-password">Password</label></td>
							<td><input type="password" id="sidebar-password" name="password" value="" size="12" class="textField" /></td>
						</tr>
						<tr>
							<td colspan="2"><input type="checkbox" id="remember" name="remember" value="1" /> <label for="remember">Remember me</label></td>
						</tr>
						<tr>
							<td colspan="2"><input type="submit" value="Login" class="button" /></td>
						</tr>
					</table>
				</form>
						</div> <div class="block" id="notification">
	<span class="blockTitle">Notifications</span>
	<ul>
					<li><a href="https://www.biotaxa.org/Phytotaxa/notification">View</a></li>
			<li><a href="https://www.biotaxa.org/Phytotaxa/notification/subscribeMailList">Subscribe</a></li>
			</ul>
</div>

<div class="block" id="sidebarNavigation">
	<span class="blockTitle">Journal Content</span>

	<form id="simpleSearchForm" action="https://www.biotaxa.org/Phytotaxa/search/search">
		<table id="simpleSearchInput">
			<tr>
				<td>
													<label for="simpleQuery">Search <br />
					<input type="text" id="simpleQuery" name="simpleQuery" size="15" maxlength="255" value="" class="textField" /></label>
								</td>
			</tr>
			<tr>
				<td><label for="searchField">
				Search Scope
				<br />
				<select id="searchField" name="searchField" size="1" class="selectMenu">
					<option label="All" value="query">All</option>
<option label="Authors" value="authors">Authors</option>
<option label="Title" value="title">Title</option>
<option label="Abstract" value="abstract">Abstract</option>
<option label="Index terms" value="indexTerms">Index terms</option>
<option label="Full Text" value="galleyFullText">Full Text</option>

				</select></label>
				</td>
			</tr>
			<tr>
				<td><input type="submit" value="Search" class="button" /></td>
			</tr>
		</table>
	</form>

	<br />

		<span class="blockSubtitle">Browse</span>
	<ul>
		<li><a href="https://www.biotaxa.org/Phytotaxa/issue/archive">By Issue</a></li>
		<li><a href="https://www.biotaxa.org/Phytotaxa/search/authors">By Author</a></li>
		<li><a href="https://www.biotaxa.org/Phytotaxa/search/titles">By Title</a></li>
		
					<li><a href="https://www.biotaxa.org/index">Other Journals</a></li>
						</ul>
	</div>
<div class="block" id="sidebarKeywordCloud">
	<span class="blockTitle">Keywords</span>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Algae"><span style="font-size: 92%;">Algae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Andes"><span style="font-size: 78%;">Andes</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Asteraceae"><span style="font-size: 79%;">Asteraceae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Atlantic%20Forest"><span style="font-size: 78%;">Atlantic Forest</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Basidiomycota"><span style="font-size: 78%;">Basidiomycota</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Brazil"><span style="font-size: 85%;">Brazil</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Bryophytes"><span style="font-size: 81%;">Bryophytes</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Cerrado"><span style="font-size: 77%;">Cerrado</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=China"><span style="font-size: 89%;">China</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Compositae"><span style="font-size: 82%;">Compositae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Eudicots"><span style="font-size: 175%;">Eudicots</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Fabaceae"><span style="font-size: 78%;">Fabaceae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Fungi"><span style="font-size: 105%;">Fungi</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=General"><span style="font-size: 78%;">General</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=ITS"><span style="font-size: 80%;">ITS</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=IUCN"><span style="font-size: 78%;">IUCN</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=India"><span style="font-size: 82%;">India</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Iran"><span style="font-size: 77%;">Iran</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Magnoliids"><span style="font-size: 80%;">Magnoliids</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Mexico"><span style="font-size: 78%;">Mexico</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Monocots"><span style="font-size: 118%;">Monocots</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Morphology"><span style="font-size: 77%;">Morphology</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Neotropics"><span style="font-size: 81%;">Neotropics</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Nomenclature"><span style="font-size: 79%;">Nomenclature</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Orchidaceae"><span style="font-size: 79%;">Orchidaceae</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Phylogeny"><span style="font-size: 79%;">Phylogeny</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Pteridophytes"><span style="font-size: 83%;">Pteridophytes</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=SEM"><span style="font-size: 78%;">SEM</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=South%20America"><span style="font-size: 80%;">South America</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Taxonomy"><span style="font-size: 92%;">Taxonomy</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Turkey"><span style="font-size: 79%;">Turkey</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=Vietnam"><span style="font-size: 78%;">Vietnam</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=biodiversity"><span style="font-size: 79%;">biodiversity</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=conservation"><span style="font-size: 79%;">conservation</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=diatoms"><span style="font-size: 78%;">diatoms</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=distribution"><span style="font-size: 78%;">distribution</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=endemic"><span style="font-size: 81%;">endemic</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=endemism"><span style="font-size: 83%;">endemism</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=flora"><span style="font-size: 78%;">flora</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=lectotype"><span style="font-size: 81%;">lectotype</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=lectotypification"><span style="font-size: 80%;">lectotypification</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=molecular%20phylogeny"><span style="font-size: 80%;">molecular phylogeny</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=morphology"><span style="font-size: 92%;">morphology</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=new%20combination"><span style="font-size: 78%;">new combination</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=new%20species"><span style="font-size: 101%;">new species</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=new%20taxon"><span style="font-size: 78%;">new taxon</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=nomenclature"><span style="font-size: 91%;">nomenclature</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=phylogeny"><span style="font-size: 91%;">phylogeny</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=systematics"><span style="font-size: 79%;">systematics</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=taxonomy"><span style="font-size: 162%;">taxonomy</span></a>
			<a href="https://www.biotaxa.org/Phytotaxa/search?subject=typification"><span style="font-size: 83%;">typification</span></a>
	</div>
<!-- Add javascript required for font sizer -->
<script type="text/javascript">
	<!--
	$(function(){
		fontSize("#sizer", "body", 9, 16, 32, ""); // Initialize the font sizer
	});
	// -->
</script>

<div class="block" id="sidebarFontSize" style="margin-bottom: 4px;">
	<span class="blockTitle">Font Size</span>
	<div id="sizer"></div>
</div>
<br /><div class="block" id="sidebarInformation">
	<span class="blockTitle">Information</span>
	<ul>
		<li><a href="https://www.biotaxa.org/Phytotaxa/information/readers">For Readers</a></li>		<li><a href="https://www.biotaxa.org/Phytotaxa/information/authors">For Authors</a></li>		<li><a href="https://www.biotaxa.org/Phytotaxa/information/librarians">For Librarians</a></li>	</ul>
</div>


<div class="block" id="sidebarRTArticleTools">

	<span class="blockTitle">Article Tools</span>
				<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/printArticle.png" class="articleToolIcon" alt="" /> <a href="javascript:openRTWindow('https://www.biotaxa.org/Phytotaxa/rt/printerFriendly/phytotaxa.383.1.4/0');">Print this article</a>
		</div>
				<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/metadata.png" class="articleToolIcon" alt=""/> <a href="javascript:openRTWindow('https://www.biotaxa.org/Phytotaxa/rt/metadata/phytotaxa.383.1.4/0');">Indexing metadata</a><br />
		</div>
				<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/citeArticle.png" class="articleToolIcon" alt=""/> <a href="javascript:openRTWindow('https://www.biotaxa.org/Phytotaxa/rt/captureCite/phytotaxa.383.1.4/0');">How to cite item</a><br />
		</div>
					<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/findingReferences.png" class="articleToolIcon" alt=""/> <a href="javascript:openRTWindow('https://www.biotaxa.org/Phytotaxa/rt/findingReferences/37161/0');">Finding References</a>
		</div>
					<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/emailArticle.png" class="articleToolIcon" alt=""/>
			Email this article <span style="font-size: 0.8em">(Login required)</span>		</div>
				<div class="articleToolItem">
			<img src="https://www.biotaxa.org/plugins/blocks/readingTools/icons/emailArticle.png" class="articleToolIcon" alt=""/>
			Email the author <span style="font-size: 0.8em">(Login required)</span>		</div>
		</div>


<script type="text/javascript">
	initRelatedItems();
</script>


<div class="block" id="sidebarRTRelatedItems">
	<span class="blockTitle">Related Items</span>
		<table width="100%">
			<tr>
				<td valign="top" style="width:26px;">
					<img src="https://www.biotaxa.org/plugins/blocks/relatedItems/icons/lookupTerms.png" class="articleToolIcon" alt=""/>
				</td>
				<td valign="top">
					<div id="relatedItems">
						<ul class="plain">
																					<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/445');">Author&#039;s work</a></li>
																																									<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/447');">Related studies</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/448');">Book searches</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/449');">Pay-per-view</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/450');">Relevant portals</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/451');">Online forums</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/452');">Teaching files</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/453');">Government policy</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/454');">Media reports</a></li>
																												<li><a href="javascript:openRTWindowWithToolbar('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/455');">Web search</a></li>
																			</ul>
					</div>
					<div id="toggleRelatedItems">
						<a id="hideRelatedItems" style="display:none;" href="javascript:void(0)">
							<img src="https://www.biotaxa.org/plugins/blocks/relatedItems/icons/magnifier_zoom_out.png" alt=""/> 
							Hide
						</a>
						<a id="showRelatedItems" href="javascript:void(0)">
							<img src="https://www.biotaxa.org/plugins/blocks/relatedItems/icons/magnifier_zoom_in.png" alt=""/>
							Show all
						</a>
					</div>
				</td>
			</tr>
		</table>
</div>


<div class="block" id="sidebarRTAuthorBios">
	<span class="blockTitle">
					About The Authors
			</span>
		<div class="authorBio">
	<p>
		<em>WEI-GUO ZHOU</em>
								<br />CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China		<br />China	</p>

	<p></p>
	</div>
	<div class="separator"></div>
		<div class="authorBio">
	<p>
		<em>DE-WEN DING</em>
								<br />CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
The First Institute of Oceanography, State Oceanic Administration (SOA), Qingdao, 266061, China		<br />China	</p>

	<p></p>
	</div>
	<div class="separator"></div>
		<div class="authorBio">
	<p>
		<em>QING-SONG YANG</em>
								<br />CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China			</p>

	<p></p>
	</div>
	<div class="separator"></div>
		<div class="authorBio">
	<p>
		<em>MANZOOR AHMAD</em>
								<br />CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China		<br />Pakistan	</p>

	<p></p>
	</div>
	<div class="separator"></div>
		<div class="authorBio">
	<p>
		<em>YUAN-ZHOU ZHANG</em>
								<br />CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China		<br />China	</p>

	<p></p>
	</div>
	<div class="separator"></div>
		<div class="authorBio">
	<p>
		<em>XIAN-CHENG LIN</em>
								<br />CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
University of Chinese Academy of Sciences, Beijing,100049, China		<br />China	</p>

	<p></p>
	</div>
	<div class="separator"></div>
		<div class="authorBio">
	<p>
		<em>YAN-YING ZHANG</em>
								<br />CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
Tropical Marine Biological Research Station in Hainan,Chinese Academy of Sciences, Sanya, 572000, China		<br />China	</p>

	<p></p>
	</div>
	<div class="separator"></div>
		<div class="authorBio">
	<p>
		<em>JUAN LING</em>
								<br />CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China		<br />China	</p>

	<p></p>
	</div>
	<div class="separator"></div>
		<div class="authorBio">
	<p>
		<em>JUN-DE DONG</em>
								<br />CAS Key Laboratory of Tropical Marine Bio-resources and Ecology; Guangdong Provincial Key Laboratory of Applied Marine Biology, South China Sea Institute of Oceanology,Guangzhou, 510301, China
Tropical Marine Biological Research Station in Hainan,Chinese Academy of Sciences, Sanya, 572000, China		<br />China	</p>

	<p></p>
	</div>
	
	</div>
			</div>
			</div>

<div id="main">

<div id="navbar">
	<ul class="menu">
		<li id="home"><a href="https://www.biotaxa.org/Phytotaxa/index">Home</a></li>
		<li id="about"><a href="https://www.biotaxa.org/Phytotaxa/about">About</a></li>

					<li id="login"><a href="https://www.biotaxa.org/Phytotaxa/login">Login</a></li>
							<li id="register"><a href="https://www.biotaxa.org/Phytotaxa/user/register">Register</a></li>
												<li id="search"><a href="https://www.biotaxa.org/Phytotaxa/search">Search</a></li>
		
					<li id="current"><a href="https://www.biotaxa.org/Phytotaxa/issue/current">Current</a></li>
			<li id="archives"><a href="https://www.biotaxa.org/Phytotaxa/issue/archive">Archives</a></li>
		
					<li id="announcements"><a href="https://www.biotaxa.org/Phytotaxa/announcement">Announcements</a></li>
				

									<li class="navItem" id="navItem-0"><a href="http://www.mapress.com/phytotaxa/">Publisher</a></li>
												<li class="navItem" id="navItem-1"><a href="http://www.biotaxa.org">Biotaxa</a></li>
						</ul>
</div>
<div id="breadcrumb">
	<a href="https://www.biotaxa.org/Phytotaxa/index" target="_parent">Home</a> &gt;
	<a href="https://www.biotaxa.org/Phytotaxa/issue/view/phytotaxa.383.1" target="_parent">Vol 383, No 1</a> &gt;	<a href="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.383.1.4/0" class="current" target="_parent">ZHOU</a>
</div>

<div id="content">


	<div id="topBar">
							<div id="accessKey">
				<img src="https://www.biotaxa.org/lib/pkp/templates/images/icons/fulltext_open_medium.gif" alt="Open Access" />
				Open Access&nbsp;
				<img src="https://www.biotaxa.org/lib/pkp/templates/images/icons/fulltext_restricted_medium.gif" alt="Restricted Access" />
									Subscription Access
							</div>
			</div>
		
	<div id="articleTitle"><h3><p><strong><em>Marileptolyngbya sina</em></strong><strong> <em>gen. nov., sp. nov.</em> and <em>Salileptolyngbya diazotrophicum gen. nov., sp. nov.</em> (Synechococcales, Cyanobacteria), species of cyanobacteria isolated from a marine ecosystem</strong></p></h3></div>
	<div id="authorString"><em>WEI-GUO ZHOU, DE-WEN DING, QING-SONG YANG, MANZOOR AHMAD, YUAN-ZHOU ZHANG, XIAN-CHENG LIN, YAN-YING ZHANG, JUAN LING, JUN-DE DONG</em></div>
	<br />
			<div id="articleAbstract">
		<h4>Abstract</h4>
		<br />
		<div><p>We isolated two filamentous, non-heterocystous cyanobacteria, namely SCSIO T-2 from Xincun Bay, Hainan Island and SCSIO 43686 from the northern South China Sea, China. The taxonomic positions of the studied strains were determined by a polyphasic approach that used morphological, molecular, and ecological analyses. The two cyanobacterial strains presented different morphology regarding their cell size and sheaths. Sequencing of the 16S rRNA gene of SCSIO T-2 showed 95% similarity with <em>Leptolyngbya </em>sp. CENA 134 (HQ730083) and SCSIO 43686 displayed 97% similarity with<em> Leptolyngbya </em>sp. L21-BG-2 (KJ206339). In addition, the<em> nifH</em> gene sequence analysis of SCSIO T-2 showed 91% similarity with <em>Lyngbya lagerheimii</em> UTEX 1930 (L15550), while SCSIO 43686 had an 88% similarity with <em>Leptolyngbya minuta</em> LEGE 07181 (KC256775). Both strains’ secondary structures of D1-D1’, Box-B, and V3 regions within the 16S–23S internal transcribed spacer (ITS) region were quite distinct from their closely related cyanobacterial strains. In addition, compared with SCSIO 43686, the SCSIO T-2 strain displayed relatively high light utilization yet low sensitivity to light intensity. Based on these characteristics, the name <em>Marileptolyngbya sina</em> SCSIO T-2 is described, and a new genus named <em>Salileptolyngbya </em>is erected to encompass a monophyletic group. We propose <em>Salileptolyngbya diazotrophicum</em> SCSIO 43686 for the type species of this new genus.</p></div>
		<br />
		</div>
	
			<div id="articleSubject">
		<h4>Keywords</h4>
		<br />
		<div>cyanobacteria, ecology, internal transcribed spacer, morphology, nifH, Algae</div>
		<br />
		</div>
	
				
			<div id="articleFullText">
		<h4>Full Text:</h4>
									<a href="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.383.1.4/37618" class="file" target="_parent">Supplemental file (ZIP)</a>
											<a href="https://www.biotaxa.org/Phytotaxa/article/view/phytotaxa.383.1.4/37638" class="file" target="_parent">PDF/A (5MB)</a>
																			<img class="accessLogo" src="https://www.biotaxa.org/lib/pkp/templates/images/icons/fulltext_restricted_medium.gif" alt="Restricted Access" />
											</div>
	
			<div id="articleCitations">
		<h4>References</h4>
		<br />
		<div>
							<p><p>Berman-Frank, I., Lundgren, P. &amp; Falkowski, P. (2003) Nitrogen fixation and photosynthetic oxygen evolution in cyanobacteria. <em>Research in microbiology</em> 154: 157–164.</p><p> https://doi.org/10.1016/S0923-2508(03)00029-9</p><p>Bhattacharyya, S., Chandrababunaidu, M.M., Sen, D., Panda, A., Ghorai, A., Bhan, S., Sanghi, N. &amp; Tripathy, S. (2015) Draft genome sequence of exopolysaccharide-producing cyanobacterium Aphanocapsa montana BDHKU 210001. <em>Genome announcements</em> 3: e00057-15.</p><p> https://doi.org/10.1128/genomeA.00057-15</p><p>Boyer, S.L., Flechtner, V.R. &amp; Johansen, J.R. (2001) Is the 16S–23S rRNA internal transcribed spacer region a good tool for use in molecular systematics and population genetics? A case study in cyanobacteria. <em>Molecular Biology and Evolution</em> 18: 1057–1069.</p><p>Casamatta, D.A., Johansen, J.R., Vis, M.L. &amp; Broadwater, S.T. (2005) Molecular and morphological characterization of ten polar and near-polar strains within the Oscillatoriales (Cyanobacteria). <em>Journal of Phycology</em> 41: 421–438.</p><p> https://doi.org/10.1111/j.1529-8817.2005.04062.x</p><p>Castenholz, R.W. (2001) General characteristics of the cyanobacteria. <em>In</em>: Boone, D.R. &amp; Castenholz, R.W. (Eds.) <em>Bergey’s Manual of Systematics of Archaea and Bacteria</em>. Springer, New York, pp. 474–487.</p><p>da Silva Malone, C.F., Rigonato, J., Laughinghouse, H.D., Schmidt, E.C., Bouzon, Z.L., Wilmotte, A., Fiore, M.F. &amp; Sant’Anna, C.L. (2015) <em>Cephalothrix </em>gen. nov. (Cyanobacteria): towards an intraspecific phylogenetic evaluation by multilocus analyses. <em>Int J Syst Evol Microbiol</em> 65: 2993–3007.</p><p> https://doi.org/10.1099/ijs.0.000369</p><p>Dadheech, P.K., Mahmoud, H., Kotut, K. &amp; Krienitz, L. (2012) <em>Haloleptolyngbya alcalis</em> gen. et sp nov., a new filamentous cyanobacterium from the soda lake Nakuru, Kenya. <em>Hydrobiologia</em> 691: 269–283.</p><p> https://doi.org/10.1007/s10750-012-1080-6</p><p>Darty, K., Denise, A. &amp; Ponty, Y. (2009) VARNA: Interactive drawing and editing of the RNA secondary structure. <em>Bioinformatics</em> 25: 1974–1975.</p><p> https://doi.org/10.1093/bioinformatics/btp250</p><p>Desikachary, T. (1959) <em>Cyanophyta. </em>ICAR Monographs on Algae, ICAR, New Delhi, 686 pp.</p><p>Engene, N., Paul, V.J., Byrum, T., Gerwick, W.H., Thor, A. &amp; Ellisman, M.H. (2013) Five chemically rich species of tropical marine cyanobacteria of the genus <em>Okeania</em> gen. nov (Oscillatoriales, Cyanoprokaryota). <em>Journal of Phycology</em> 49: 1095–1106.</p><p> https://doi.org/10.1111/jpy.12115</p><p>Fang, W., Guo, P., Liu, C., Fang, G. &amp; Li, S. (2015) Observed sub-inertial current variability and volume transport over the continental shelf in the northern South China Sea. <em>Estuarine Coastal and Shelf Science</em> 157: 19–31.</p><p> https://doi.org/10.1016/j.ecss.2015.02.001</p><p>Ferris, M.J. &amp; Palenik, B. (1998) Niche adaptation in ocean cyanobacteria. <em>Nature</em> 396: 226–228.</p><p>Foster, J.S., Green, S.J., Ahrendt, S.R., Golubic, S., Reid, R.P., Hetherington, K.L. &amp; Bebout, L. (2009) Molecular and morphological characterization of cyanobacterial diversity in the stromatolites of Highborne Cay, Bahamas. <em>Isme Journal</em> 3: 573–587.</p><p> https://doi.org/10.1038/ismej.2008.129</p><p>Foster, R.A. &amp; Zehr, J.P. (2010) Characterization of diatom-cyanobacteria symbioses on the basis of nifH, hetR and 16S rRNA sequences. <em>Environmental Microbiology</em> 8: 1913–1925.</p><p> https://doi.org/10.1111/j.1462-2920.2006.01068.x</p><p>Fox, G.E., Wisotzkey, J.D. &amp; Jurtshuk JR, P. (1992) How close is close: 16S rRNA sequence identity may not be sufficient to guarantee species identity. <em>International Journal of Systematic and Evolutionary Microbiology</em> 42: 166–170.</p><p> https://doi.org/10.1099/00207713-42-1-166</p><p>Han, K.S. &amp; Byun, Y. (2003) PseudoViewer2: visualization of RNA pseudoknots of any type. <em>Nucleic Acids Research</em> 31: 3432–3440.</p><p> https://doi.org/10.1093/nar/gkg539</p><p>Iteman, I., Rippka, R., de Marsac, N.T. &amp; Herdman, M. (2000) Comparison of conserved structural and regulatory domains within divergent 16S rRNA-23S rRNA spacer sequences of cyanobacteria. <em>Microbiology-Sgm</em> 146: 1275–1286.</p><p> https://doi.org/10.1099/00221287-146-6-1275</p><p>Johansen, J.R., Kovacik, L., Casamatta, D.A., Iková, K.F. &amp; Kaštovský, J. (2011) Utility of 16S-23S ITS sequence and secondary structure for recognition of intrageneric and intergeneric limits within cyanobacterial taxa: <em>Leptolyngbya corticola</em> sp. nov.(Pseudanabaenaceae, Cyanobacteria). <em>Nova Hedwigia</em> 92: 283–302.</p><p> https://doi.org/10.1127/0029-5035/2011/0092-0283</p><p>Kim, O.S., Cho, Y.J., Lee, K., Yoon, S.H., Kim, M., Na, H., Park, S.C., Jeon, Y.S., Lee, J.H. &amp; Yi, H. (2012) Introducing EzTaxon-e: a prokaryotic 16S rRNA gene sequence database with phylotypes that represent uncultured species. <em>International journal of systematic and evolutionary microbiology</em> 62: 716–721.</p><p> https://doi.org/10.1099/ijs.0.038075-0</p><p>Komárek, J. (2007) Phenotype diversity of the cyanobacterial genus <em>Leptolyngbya</em> in the maritime Antarctic. <em>Pol Polar Res</em> 28: 211–231.</p><p>Komárek, J. (2016) A polyphasic approach for the taxonomy of cyanobacteria: principles and applications. <em>European Journal of Phycology</em> 51: 346–353.</p><p> https://doi.org/10.1080/09670262.2016.1163738</p><p>Komarek, J. &amp; Anagnostidis, K. (1986) Modern approach to the classification system of cyanophytes. 2-Chroococcales. <em>Arch. Hydrobiol. Suppl.</em> 73: 157–226.</p><p>Komárek, J. &amp; Anagnostidis, K. (2005) Cyanoprokaryota, Part 2: Oscillatoriales. <em>In</em>: Büdel, B., Krienitz, L., Gärtner, G. &amp; Schagerl, M. (Eds.) <em>Süsswasserflora von Mitteleuropa</em>. Elsevier Spektrum Akademischer Verlag, München, pp. 1–759.</p><p>Komárek, J., Kaštovský, J., Mareš, J. &amp; Johansen, J. (2014) Taxonomic classification of cyanoprokaryotes (cyanobacterial genera) 2014, using a polyphasic approach. <em>Preslia</em> 86: 295–335.</p><p>Komárek, J., Kaštovský, J., Ventura, S., Turicchia, S. &amp; Šmarda, J. (2009) The cyanobacterial genus Phormidesmis. <em>Algological Studies</em> 129: 41–59.</p><p>Lane, D.J. (1991) 16S/23S rRNA sequencing. <em>Nucleic Acid Techniques in Bacterial Systematics</em> 125–175.</p><p>Li, X. &amp; Li, R. (2016) <em>Limnolyngbya circumcreta </em>gen. &amp; comb. nov. (Synechococcales, Cyanobacteria) with three geographical (provincial) genotypes in China. <em>Phycologia</em> 55: 478–491.</p><p> https://doi.org/10.2216/15-149.1</p><p>Li, Z. &amp; Brand, J. (2007) <em>Leptolyngbya nodulosa</em> sp nov (Oscillatoriaceae), a subtropical marine cyanobacterium that produces a unique multicellular structure. <em>Phycologia</em> 46: 396–401.</p><p> https://doi.org/10.2216/06-89.1</p><p>Lopes, V.R., Ramos, V., Martins, A., Sousa, M., Welker, M., Antunes, A. &amp; Vasconcelos, V.M. (2012) Phylogenetic, chemical and morphological diversity of cyanobacteria from Portuguese temperate estuaries. <em>Marine Environmental Research</em> 73: 7–16.</p><p> https://doi.org/10.1016/j.marenvres.2011.10.005</p><p>Marcal Vieira Vaz, M.G., Genuario, D.B., Dini Andreote, A.P., Silva Malone, C.F., Sant’Anna, C.L., Barbiero, L. &amp; Fiore, M.F. (2015) <em>Pantanalinema</em> gen. nov and <em>Alkalinema </em>gen. nov.: novel pseudanabaenacean genera (Cyanobacteria) isolated from saline-alkaline lakes. <em>International Journal of Systematic and Evolutionary Microbiology</em> 65: 298–308.</p><p> https://doi.org/10.1099/ijs.0.070110-0</p><p>Martins, M.D. &amp; Branco, L.H. (2016) <em>Potamolinea</em> gen. nov. (Oscillatoriales, Cyanobacteria): a phylogenetically and ecologically coherent cyanobacterial genus. <em>International Journal of Systematic and Evolutionary Microbiology</em> 66: 3632–3641.</p><p> https://doi.org/10.1099/ijsem.0.001243</p><p>Nubel, U., Garcia-Pichel, F. &amp; Muyzer, G. (2000) The halotolerance and phylogeny of cyanobacteria with tightly coiled trichomes (Spirulina Turpin) and the description of <em>Halospirulina tapeticola</em> gen. nov., sp nov. <em>International Journal of Systematic and Evolutionary Microbiology</em> 50: 1265–1277.</p><p> https://doi.org/10.1099/00207713-50-3-1265</p><p>Olson, J., Steppe, T., Litaker, R. &amp; Paerl, H. (1998) N2-fixing microbial consortia associated with the ice cover of Lake Bonney, Antarctica. <em>Microbial Ecology</em> 36: 231–238.</p><p> https://doi.org/10.1007/s002489900110</p><p>Osorio-Santos, K., Pietrasiak, N., Bohunicka, M., Miscoe, L.H., Kovacik, L., Martin, M.P. &amp; Johansen, J.R. (2014) Seven new species of <em>Oculatella</em> (Pseudanabaenales, Cyanobacteria): taxonomically recognizing cryptic diversification. <em>European Journal of Phycology</em> 49: 450–470.</p><p> https://doi.org/10.1080/09670262.2014.976843</p><p>Pamplona Silva, C.S., Genuario, D.B., Marcal Vieira Vaz, M.G. &amp; Fiore, M.F. (2014) Phylogeny of culturable cyanobacteria from Brazilian mangroves. <em>Systematic and Applied Microbiology</em> 37: 100–112.</p><p> https://doi.org/10.1016/j.syapm.2013.12.003</p><p>Paul, V.J., Thacker, R.W., Banks, K. &amp; Golubic, S. (2005) Benthic cyanobacterial bloom impacts the reefs of South Florida (Broward County, USA). <em>Coral Reefs</em> 24: 693–697.</p><p> https://doi.org/10.1007/s00338-005-0061-x</p><p>Perkerson, R.B., III, Johansen, J.R., Kovacik, L., Brand, J., Kastovsky, J. &amp; Casamatta, D.A. (2011) A unique Pseudanabaenalean (Cyanobacteria) genus <em>Nodosilinea</em> gen. nov based on morphological and molecular data. <em>Journal of Phycology</em> 47: 1397–1412.</p><p> https://doi.org/10.1111/j.1529-8817.2011.01077.x</p><p>Posada, D. (2008) jModelTest: Phylogenetic model averaging. <em>Molecular Biology and Evolution</em> 25: 1253–1256.</p><p> https://doi.org/10.1093/molbev/msn083</p><p>Posada, D. &amp; Crandall, K.A. (1998) MODELTEST: testing the model of DNA substitution. <em>Bioinformatics</em> 14: 817–818.</p><p> https://doi.org/10.1093/bioinformatics/14.9.817</p><p>Rehakova, K., Johansen, J.R., Casamatta, D.A., Xueson, L. &amp; Vincent, J. (2007) Morphological and molecular characterization of selected desert soil cyanobacteria: Three species new to science including Mojavia pulchra gen. et sp nov. <em>Phycologia</em> 46: 481–502.</p><p> https://doi.org/10.2216/06-92.1</p><p>Rippka, R., Deruelles, J., Waterbury, J.B., Herdman, M. &amp; Stanier, R.Y. (1979) Generic assignments, strain histories and properties of pure cultures of cyanobacteria. <em>Microbiology</em> 111: 1–61.</p><p> https://doi.org/10.1099/00221287-111-1-1</p><p>Ronquist, F., Teslenko, M., Paul, V.D.M., Ayres, D.L., Darling, A., Höhna, S., Larget, B., Liu, L., Suchard, M.A. &amp; Huelsenbeck, J.P. (2012) MrBayes 3.2: Efficient Bayesian Phylogenetic Inference and Model Choice Across a Large Model Space. <em>Systematic Biology</em> 61: 539–542.</p><p>Saitou, N. &amp; Nei, M. (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. <em>Molecular biology and evolution</em> 4: 406–425.</p><p> https://doi.org/10.1093/oxfordjournals.molbev.a040454</p><p>Schattner, P., Brooks, A.N. &amp; Lowe, T.M. (2005) The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs. <em>Nucleic Acids Research</em> 33: W686-W689.</p><p> https://doi.org/10.1093/nar/gki366</p><p>Sciuto, K. &amp; Moro, I. (2016) Detection of the new cosmopolitan genus Thermoleptolyngbya (Cyanobacteria, Leptolyngbyaceae) using the 16S rRNA gene and 16S-23S ITS region. <em>Molecular Phylogenetics and Evolution</em> 105: 15–35.</p><p> https://doi.org/10.1016/j.ympev.2016.08.010</p><p>Sharp, K., Arthur, K.E., Gu, L., Ross, C., Harrison, G., Gunasekera, S.P., Meickle, T., Matthew, S., Luesch, H., Thacker, R.W., Sherman, D.H. &amp; Paul, V.J. (2009) Phylogenetic and Chemical Diversity of Three Chemotypes of Bloom-Forming Lyngbya Species (Cyanobacteria: Oscillatoriales) from Reefs of Southeastern Florida. <em>Applied and Environmental Microbiology</em> 75: 2879–2888.</p><p> https://doi.org/10.1128/aem.02656-08</p><p>Silva-Stenico, M.E., Vieira, F.D., Genuário, D.B., Silva, C.S., Moraes, L.A.B. &amp; Fiore, M.F. (2012) Decolorization of textile dyes by cyanobacteria. <em>Journal of the Brazilian Chemical Society</em> 23: 1863–1870.</p><p>Song, G.-F., Jiang, Y.-G. &amp; Li, R.-H. (2015) <em>Scytolyngbya timoleontis</em>, gen. et sp. nov. (Leptolyngbyaceae, Cyanobacteria): a novel false branching Cyanobacteria from China. <em>Phytotaxa</em> 224: 72–84.</p><p> https://doi.org/10.11646/phytotaxa.224.1.5</p><p>Spring, S., Brinkmann, N., Murrja, M., Spröer, C., Reitner, J. &amp; Klenk, H.-P. (2015) High diversity of culturable prokaryotes in a lithifying hypersaline microbial mat. <em>Geomicrobiology Journal</em> 32: 332–346.</p><p> https://doi.org/10.1080/01490451.2014.913095</p><p>Starmach, K., Botaniki, I. &amp; Nauk, P.A. (1966) Cyanophyta-sinice, Glaucophyta-glaukofity. <em>In</em>: Starmach, K. (Eds.) <em>Flora Slodkowodna Polski</em>. Państwowe Wydawnictwo Naukowe, Warszawa, pp. 1–808.</p><p>Steindler, L., Huchon, D., Avni, A. &amp; Ilan, M. (2005) 16S rRNA phylogeny of sponge-associated cyanobacteria. <em>Appl Environ Microbiol</em> 71: 4127–4131.</p><p> https://doi.org/10.1128/AEM.71.7.4127-4131.2005</p><p>Strunecky, O., Bohunicka, M., Johansen, J.R., Capkova, K., Raabova, L., Dvorak, P. &amp; Komarek, J. (2017) A revision of the genus Geitlerinema and a description of the genus Anagnostidinema gen. nov (Oscillatoriophycidae, Cyanobacteria). <em>Fottea</em> 17: 114–126.</p><p> https://doi.org/10.5507/fot.2016.025</p><p>Tamura, K., Peterson, D., Peterson, N., Stecher, G., Nei, M. &amp; Kumar, S. (2011) MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods. <em>Molecular biology and evolution</em> 28: 2731–2739.</p><p> https://doi.org/10.1093/molbev/msr121</p><p>Tamura, K., Stecher, G., Peterson, D., Filipski, A. &amp; Kumar, S. (2013) MEGA6: molecular evolutionary genetics analysis version 6.0. <em>Molecular biology and evolution</em> 30: 2725–2729.</p><p> https://doi.org/10.1093/molbev/mst197</p><p>Taton, A., Grubisic, S., Brambilla, E., De Wit, R. &amp; Wilmotte, A. (2003) Cyanobacterial diversity in natural and artificial microbial mats of Lake Fryxell (McMurdo Dry Valleys, Antarctica): a morphological and molecular approach. <em>Applied and environmental microbiology</em> 69: 5157–5169.</p><p> https://doi.org/10.1128/AEM.69.9.5157-5169.2003</p><p>Taton, A., Grubisic, S., Ertz, D., Hodgson, D.A., Piccardi, R., Biondi, N., Tredici, M.R., Mainini, M., Losi, D., Marinelli, F. &amp; Wilmotte, A. (2006) Polyphasic study of Antarctic cyanobacterial strains. <em>Journal of Phycology</em> 42: 1257–1270.</p><p> https://doi.org/10.1111/j.1529-8817.2006.00278.x</p><p>Taton, A., Wilmotte, A., Smarda, J., Elster, J. &amp; Komarek, J. (2011) <em>Plectolyngbya hodgsonii</em>: a novel filamentous cyanobacterium from Antarctic lakes. <em>Polar Biology</em> 34: 181–191.</p><p> https://doi.org/10.1007/s00300-010-0868-y</p><p>Tsiodras, S., Gold, H.S., Sakoulas, G., Eliopoulos, G.M., Wennersten, C., Venkataraman, L., Moellering, R.C. &amp; Ferraro, M.J. (2001) Linezolid resistance in a clinical isolate of Staphylococcus aureus. <em>The Lancet</em> 358: 207–208.</p><p> https://doi.org/10.1016/S0140-6736(01)05410-1</p><p>Turner, S. (1997) Molecular systematics of oxygenic photosynthetic bacteria. <em>In</em>: Bhattacharya, D. (Eds.) <em>Origins of Algae and their Plastids</em>. Springer, Berlin, pp. 13–52.</p><p>Whitton, B.A. (1992) Diversity, ecology, and taxonomy of the cyanobacteria. <em>In</em>: Mann, N.H. &amp; Carr, N.G. (Eds.) <em>Photosynthetic prokaryotes</em>. Plenum Press, New York, pp. 1–51.</p><p>Whitton, B.A. &amp; Potts, M. (2000) <em>The ecology of cyanobacteria: their diversity in time and space. </em>Kluwer Academic Publishers, Dordrecht, the Netherands,</p><p>Wilmotte, A. (1994) Molecular evolution and taxonomy of the cyanobacteria. <em>In</em>: Bryant, D.A. (Eds.) <em>The molecular biology of cyanobacteria</em>. Kluwer Academic Publishers, Dordrecht, pp. 1–25.</p><p>Wilmotte, A. &amp; Herdman, M. (2001) Phylogenetic relationships among the cyanobacteria based on 16S rRNA sequences. <em>In</em>: Boone, D.R. &amp; Castenholz, R.W. (Eds.) <em>Bergey’s Manual of Systematics of Archaea and Bacteria</em>. Springer, New York, pp. 487–493.</p><p>Wilmotte, A., Neefs, J.M. &amp; De, W.R. (1994) Evolutionary affiliation of the marine nitrogen-fixing cyanobacterium Trichodesmium sp. strain NIBB 1067, derived by 16S ribosomal RNA sequence analysis. <em>Microbiology</em> 140 (Pt 8): 2159–2164.</p><p> https://doi.org/ 10.1099/13500872-140-8-2159</p><p>Zammit, G., Billi, D. &amp; Albertano, P. (2012) The subaerophytic cyanobacterium <em>Oculatella subterranea</em> (Oscillatoriales, Cyanophyceae) gen. et sp. nov.: a cytomorphological and molecular description. <em>European Journal of Phycology</em> 47: 341–354.</p><p> https://doi.org/10.1080/09670262.2012.717106</p><p>Zehr, J.P. (2011) Nitrogen fixation by marine cyanobacteria. <em>Trends in microbiology</em> 19: 162–173.</p><p> https://doi.org/10.1016/j.tim.2010.12.004</p><p>Zehr, J.P., Mellon, M.T. &amp; Hiorns, W.D. (1997) Phylogeny of cyanobacterial <em>nifH </em>genes: evolutionary implications and potential applications to natural assemblages. <em>Microbiology</em> 143: 1443–1450.</p><p> https://doi.org/10.1099/00221287-143-4-1443</p><p>Zuker, M. (2003) Mfold web server for nucleic acid folding and hybridization prediction. <em>Nucleic Acids Research</em> 31: 3406–3415.</p><p> https://doi.org/10.1093/nar/gkg595</p></p>
					</div>
		<br />
		</div>
	
							<br />
		<br />
		DOI: <a id="pub-id::doi" href="https://doi.org/10.11646/phytotaxa.383.1.4">https://doi.org/10.11646/phytotaxa.383.1.4</a>																																			

<div class="separator"></div>

<h3>Refbacks</h3>

<ul>
				<li>There are currently no refbacks.</li>
	</ul>


		
<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=https%3A%2F%2Fwww.biotaxa.org%2FPhytotaxa%2Farticle%2Fview%2F37161&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.title=Phytotaxa&amp;rft.jtitle=Phytotaxa&amp;rft.atitle=%3Cp%3E%3Cstrong%3E%3Cem%3EMarileptolyngbya+sina%3C%2Fem%3E%3C%2Fstrong%3E%3Cstrong%3E+%3Cem%3Egen.+nov.%2C+sp.+nov.%3C%2Fem%3E+and+%3Cem%3ESalileptolyngbya+diazotrophicum+gen.+nov.%2C+sp.+nov.%3C%2Fem%3E+%28Synechococcales%2C+Cyanobacteria%29%2C+species+of+cyanobacteria+isolated+from+a+marine+ecosystem%3C%2Fstrong%3E%3C%2Fp%3E&amp;rft.artnum=phytotaxa.383.1.4&amp;rft.stitle=Phytotaxa&amp;rft.volume=383&amp;rft.issue=1&amp;rft.aulast=ZHOU&amp;rft.aufirst=WEI-GUO&amp;rft.auinit=&amp;rft.date=2018-12-13&amp;rft.au=WEI-GUO+ZHOU&amp;rft.au=DE-WEN+DING&amp;rft.au=QING-SONG+YANG&amp;rft.au=MANZOOR+AHMAD&amp;rft.au=YUAN-ZHOU+ZHANG&amp;rft.au=XIAN-CHENG+LIN&amp;rft.au=YAN-YING+ZHANG&amp;rft.au=JUAN+LING&amp;rft.au=JUN-DE+DONG&amp;rft_id=info%3Adoi%2F10.11646%2Fphytotaxa.383.1.4&amp;rft.pages=75%E2%80%9392&amp;rft.issn=1179-3155&amp;rft.eissn=1179-3163"></span>
<!-- Google Analytics -->

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-89286081-1', 'auto');
ga('send', 'pageview');

</script>

<!-- End Google Analytics -->
<br /><br />
<p><span style="color: #0099cc; font-family: 'Times New Roman'; font-size: small;"><img src="/public/site/images/mapress/mpicon.gif" alt="" /></span></p><p><span style="color: #0099cc; font-family: 'Times New Roman'; font-size: small;">ISSN <span lang="EN-AU">1179-3155 (print); </span></span><span style="color: #0099cc; font-family: 'Times New Roman'; font-size: small;">ISSN 1179-3163 (online)</span></p><p>Published by <a title="Magnolia Press" href="http://www.mapress.com/phytotaxa/index.htm" target="_blank">Magnolia Press</a>, Auckland, New Zealand</p><div> </div>
</div><!-- content -->
</div><!-- main -->
</div><!-- body -->

<script type="text/javascript">

<!--
	// Open "Define Terms" context when double-clicking any text
	function openSearchTermWindow(url) {
		var term;
		if (window.getSelection) {
			term = window.getSelection();
		} else if (document.getSelection) {
			term = document.getSelection();
		} else if(document.selection && document.selection.createRange && document.selection.type.toLowerCase() == 'text') {
			var range = document.selection.createRange();
			term = range.text;
		}
		if (term != ""){
			if (url.indexOf('?') > -1) openRTWindowWithToolbar(url + '&defineTerm=' + term);
			else openRTWindowWithToolbar(url + '?defineTerm=' + term);
		}
	}

	if(document.captureEvents) {
		document.captureEvents(Event.DBLCLICK);
	}

	// Make sure to only open the reading tools when double clicking within the galley
	if (document.getElementById('inlinePdfResizer')) {
		context = document.getElementById('inlinePdfResizer');
	}
	else if (document.getElementById('content')) {
		context = document.getElementById('content');
	}
	else {
		context = document;
	}

	context.ondblclick = new Function("openSearchTermWindow('https://www.biotaxa.org/Phytotaxa/rt/context/phytotaxa.383.1.4/0/446')");
// -->

</script>


</div> <!-- container -->
<script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam-cell.nr-data.net","licenseKey":"4f72ad4ec1","applicationID":"416471445","transactionName":"ZF0AZkZRWRVUVEINDF0XN0BdH14IUVJOShNbSA==","queueTime":0,"applicationTime":340,"atts":"SBoDEA5LShs=","errorBeacon":"bam-cell.nr-data.net","agent":""}</script></body>
</html>